From 69738b7f8122fb04b5aaefe91c8559da9d65edce Mon Sep 17 00:00:00 2001
From: Hiroshi DOYU <Hiroshi.DOYU@nokia.com>
Date: Sat, 13 Sep 2008 11:27:33 +0300
Subject: [PATCH] BRIDGE: Add 'dspbridge' dirname in header file inclusion

To remove dspbridge from -I

for i in ~/lk/arch/arm/plat-omap/include/dspbridge/*.h
do
	j=$(basename $i)
	echo $j
	sed -i -e 's@<\('"$j"'\)>@<dspbridge/\1>@g' \
	drivers/dsp/bridge/*/*.[ch] \
	arch/arm/plat-omap/include/dspbridge/*.h
done

ex)
-#include <wmdioctl.h>
-#include "clk.h"
+#include <dspbridge/wmdioctl.h>
+#include <dspbridge/clk.h>

Signed-off-by: Hiroshi DOYU <Hiroshi.DOYU@nokia.com>
---
 arch/arm/plat-omap/include/dspbridge/_chnl_sm.h    |   10 ++--
 arch/arm/plat-omap/include/dspbridge/_dcd.h        |    2 +-
 arch/arm/plat-omap/include/dspbridge/cfg.h         |    4 +-
 arch/arm/plat-omap/include/dspbridge/chnl.h        |    2 +-
 arch/arm/plat-omap/include/dspbridge/chnl_sm.h     |    2 +-
 arch/arm/plat-omap/include/dspbridge/chnlpriv.h    |    6 +-
 arch/arm/plat-omap/include/dspbridge/cmm.h         |    6 +-
 arch/arm/plat-omap/include/dspbridge/cmmdefs.h     |    2 +-
 arch/arm/plat-omap/include/dspbridge/cod.h         |    2 +-
 arch/arm/plat-omap/include/dspbridge/csl.h         |    2 +-
 arch/arm/plat-omap/include/dspbridge/dbc.h         |    4 +-
 arch/arm/plat-omap/include/dspbridge/dbdcd.h       |    6 +-
 arch/arm/plat-omap/include/dspbridge/dbdcddef.h    |    4 +-
 arch/arm/plat-omap/include/dspbridge/dbdefs.h      |    6 +-
 arch/arm/plat-omap/include/dspbridge/dbg.h         |    2 +-
 arch/arm/plat-omap/include/dspbridge/dbl.h         |    4 +-
 arch/arm/plat-omap/include/dspbridge/dbll.h        |    4 +-
 arch/arm/plat-omap/include/dspbridge/dehdefs.h     |    4 +-
 arch/arm/plat-omap/include/dspbridge/dev.h         |   20 +++---
 arch/arm/plat-omap/include/dspbridge/disp.h        |    8 +-
 arch/arm/plat-omap/include/dspbridge/dmm.h         |    2 +-
 arch/arm/plat-omap/include/dspbridge/drv.h         |    4 +-
 arch/arm/plat-omap/include/dspbridge/gb.h          |    2 +-
 arch/arm/plat-omap/include/dspbridge/gh.h          |    2 +-
 arch/arm/plat-omap/include/dspbridge/gt.h          |    2 +-
 arch/arm/plat-omap/include/dspbridge/host_os.h     |    2 +-
 arch/arm/plat-omap/include/dspbridge/io.h          |    6 +-
 arch/arm/plat-omap/include/dspbridge/io_sm.h       |    6 +-
 arch/arm/plat-omap/include/dspbridge/list.h        |    2 +-
 arch/arm/plat-omap/include/dspbridge/mem.h         |    4 +-
 arch/arm/plat-omap/include/dspbridge/mgr.h         |    2 +-
 arch/arm/plat-omap/include/dspbridge/msg.h         |    4 +-
 arch/arm/plat-omap/include/dspbridge/nldr.h        |   10 ++--
 arch/arm/plat-omap/include/dspbridge/nldrdefs.h    |    4 +-
 arch/arm/plat-omap/include/dspbridge/node.h        |    8 +-
 arch/arm/plat-omap/include/dspbridge/nodepriv.h    |    6 +-
 arch/arm/plat-omap/include/dspbridge/proc.h        |    4 +-
 arch/arm/plat-omap/include/dspbridge/pwr.h         |    6 +-
 arch/arm/plat-omap/include/dspbridge/pwr_sh.h      |    2 +-
 .../plat-omap/include/dspbridge/resourcecleanup.h  |    4 +-
 arch/arm/plat-omap/include/dspbridge/rms_sh.h      |    2 +-
 arch/arm/plat-omap/include/dspbridge/rmstypes.h    |    2 +-
 arch/arm/plat-omap/include/dspbridge/services.h    |    2 +-
 arch/arm/plat-omap/include/dspbridge/std.h         |    2 +-
 arch/arm/plat-omap/include/dspbridge/strm.h        |    4 +-
 arch/arm/plat-omap/include/dspbridge/util.h        |    2 +-
 arch/arm/plat-omap/include/dspbridge/wcdioctl.h    |    8 +-
 arch/arm/plat-omap/include/dspbridge/wmd.h         |   14 ++--
 arch/arm/plat-omap/include/dspbridge/wmddeh.h      |    4 +-
 arch/arm/plat-omap/include/dspbridge/wmdio.h       |    4 +-
 arch/arm/plat-omap/include/dspbridge/wmdmsg.h      |    2 +-
 drivers/dsp/bridge/Kbuild                          |    2 -
 drivers/dsp/bridge/dynload/dlclasses_hdr.h         |    2 +-
 drivers/dsp/bridge/dynload/getsection.c            |    2 +-
 drivers/dsp/bridge/dynload/header.h                |    2 +-
 drivers/dsp/bridge/gen/_gt_para.c                  |    6 +-
 drivers/dsp/bridge/gen/gb.c                        |    6 +-
 drivers/dsp/bridge/gen/gh.c                        |    8 +-
 drivers/dsp/bridge/gen/gs.c                        |    8 +-
 drivers/dsp/bridge/gen/gt.c                        |    4 +-
 drivers/dsp/bridge/gen/uuidutil.c                  |   10 ++--
 drivers/dsp/bridge/pmgr/chnl.c                     |   32 +++++-----
 drivers/dsp/bridge/pmgr/chnlobj.h                  |    4 +-
 drivers/dsp/bridge/pmgr/cmm.c                      |   26 ++++----
 drivers/dsp/bridge/pmgr/cod.c                      |   24 ++++----
 drivers/dsp/bridge/pmgr/dbl.c                      |   22 +++---
 drivers/dsp/bridge/pmgr/dbll.c                     |   26 ++++----
 drivers/dsp/bridge/pmgr/dev.c                      |   52 ++++++++--------
 drivers/dsp/bridge/pmgr/dmm.c                      |   24 ++++----
 drivers/dsp/bridge/pmgr/io.c                       |   22 +++---
 drivers/dsp/bridge/pmgr/ioobj.h                    |    4 +-
 drivers/dsp/bridge/pmgr/msg.c                      |   22 +++---
 drivers/dsp/bridge/pmgr/msgobj.h                   |    4 +-
 drivers/dsp/bridge/pmgr/wcd.c                      |   52 ++++++++--------
 drivers/dsp/bridge/rmgr/dbdcd.c                    |   24 ++++----
 drivers/dsp/bridge/rmgr/disp.c                     |   34 +++++-----
 drivers/dsp/bridge/rmgr/drv.c                      |   40 ++++++------
 drivers/dsp/bridge/rmgr/drv_interface.c            |   48 +++++++-------
 drivers/dsp/bridge/rmgr/dspdrv.c                   |   32 +++++-----
 drivers/dsp/bridge/rmgr/mgr.c                      |   26 ++++----
 drivers/dsp/bridge/rmgr/nldr.c                     |   32 +++++-----
 drivers/dsp/bridge/rmgr/node.c                     |   68 ++++++++++----------
 drivers/dsp/bridge/rmgr/proc.c                     |   56 ++++++++--------
 drivers/dsp/bridge/rmgr/pwr.c                      |   12 ++--
 drivers/dsp/bridge/rmgr/rmm.c                      |   16 ++--
 drivers/dsp/bridge/rmgr/strm.c                     |   30 ++++----
 drivers/dsp/bridge/services/cfg.c                  |   20 +++---
 drivers/dsp/bridge/services/clk.c                  |   20 +++---
 drivers/dsp/bridge/services/csl.c                  |   12 ++--
 drivers/dsp/bridge/services/dbg.c                  |   12 ++--
 drivers/dsp/bridge/services/dpc.c                  |   16 ++--
 drivers/dsp/bridge/services/kfile.c                |   18 +++---
 drivers/dsp/bridge/services/list.c                 |   12 ++--
 drivers/dsp/bridge/services/mem.c                  |   16 ++--
 drivers/dsp/bridge/services/ntfy.c                 |   22 +++---
 drivers/dsp/bridge/services/reg.c                  |   20 +++---
 drivers/dsp/bridge/services/regsup.c               |   18 +++---
 drivers/dsp/bridge/services/services.c             |   36 +++++-----
 drivers/dsp/bridge/services/sync.c                 |   18 +++---
 drivers/dsp/bridge/wmd/_deh.h                      |    6 +-
 drivers/dsp/bridge/wmd/_msg_sm.h                   |    4 +-
 drivers/dsp/bridge/wmd/_tiomap.h                   |    8 +-
 drivers/dsp/bridge/wmd/chnl_sm.c                   |   28 ++++----
 drivers/dsp/bridge/wmd/io_sm.c                     |   46 +++++++-------
 drivers/dsp/bridge/wmd/mmu_fault.c                 |   22 +++---
 drivers/dsp/bridge/wmd/msg_sm.c                    |   20 +++---
 drivers/dsp/bridge/wmd/tiomap3430.c                |   50 +++++++-------
 drivers/dsp/bridge/wmd/tiomap3430_pwr.c            |   26 ++++----
 drivers/dsp/bridge/wmd/tiomap_io.c                 |   18 +++---
 drivers/dsp/bridge/wmd/tiomap_sm.c                 |   24 ++++----
 drivers/dsp/bridge/wmd/ue_deh.c                    |   32 +++++-----
 111 files changed, 766 insertions(+), 768 deletions(-)

--- a/arch/arm/plat-omap/include/dspbridge/_chnl_sm.h
+++ b/arch/arm/plat-omap/include/dspbridge/_chnl_sm.h
@@ -55,12 +55,12 @@
 #ifndef _CHNL_SM_
 #define _CHNL_SM_
 
-#include <wcd.h>
-#include <wmd.h>
-#include <dpc.h>
+#include <dspbridge/wcd.h>
+#include <dspbridge/wmd.h>
+#include <dspbridge/dpc.h>
 
-#include <list.h>
-#include <ntfy.h>
+#include <dspbridge/list.h>
+#include <dspbridge/ntfy.h>
 
 /*
  *  These target side symbols define the beginning and ending addresses
--- a/arch/arm/plat-omap/include/dspbridge/_dcd.h
+++ b/arch/arm/plat-omap/include/dspbridge/_dcd.h
@@ -54,7 +54,7 @@
 #ifndef _WCD_
 #define _WCD_
 
-#include <wcdioctl.h>
+#include <dspbridge/wcdioctl.h>
 
 /*
  *  ======== WCD_CallDevIOCtl ========
--- a/arch/arm/plat-omap/include/dspbridge/cfg.h
+++ b/arch/arm/plat-omap/include/dspbridge/cfg.h
@@ -64,8 +64,8 @@
 
 #ifndef CFG_
 #define CFG_
-#include <host_os.h>
-#include <cfgdefs.h>
+#include <dspbridge/host_os.h>
+#include <dspbridge/cfgdefs.h>
 
 /*
  *  ======== CFG_Exit ========
--- a/arch/arm/plat-omap/include/dspbridge/chnl.h
+++ b/arch/arm/plat-omap/include/dspbridge/chnl.h
@@ -57,7 +57,7 @@
 #ifndef CHNL_
 #define CHNL_
 
-#include <chnlpriv.h>
+#include <dspbridge/chnlpriv.h>
 
 /*
  *  ======== CHNL_AddIOReq ========
--- a/arch/arm/plat-omap/include/dspbridge/chnl_sm.h
+++ b/arch/arm/plat-omap/include/dspbridge/chnl_sm.h
@@ -50,7 +50,7 @@
 #ifndef CHNLSM_
 #define CHNLSM_
 
-#include <wmd.h>
+#include <dspbridge/wmd.h>
 
 /*
  *  ======== CHNLSM_DisableInterrupt ========
--- a/arch/arm/plat-omap/include/dspbridge/chnlpriv.h
+++ b/arch/arm/plat-omap/include/dspbridge/chnlpriv.h
@@ -46,9 +46,9 @@
 #ifndef CHNLPRIV_
 #define CHNLPRIV_
 
-#include <chnldefs.h>
-#include <devdefs.h>
-#include <sync.h>
+#include <dspbridge/chnldefs.h>
+#include <dspbridge/devdefs.h>
+#include <dspbridge/sync.h>
 
 /* CHNL Object validation signatures: */
 #define CHNL_MGRSIGNATURE   0x52474D43	/* "CMGR" (in reverse). */
--- a/arch/arm/plat-omap/include/dspbridge/cmm.h
+++ b/arch/arm/plat-omap/include/dspbridge/cmm.h
@@ -67,10 +67,10 @@
 #ifndef CMM_
 #define CMM_
 
-#include <devdefs.h>
+#include <dspbridge/devdefs.h>
 
-#include <cmmdefs.h>
-#include <host_os.h>
+#include <dspbridge/cmmdefs.h>
+#include <dspbridge/host_os.h>
 
 /*
  *  ======== CMM_CallocBuf ========
--- a/arch/arm/plat-omap/include/dspbridge/cmmdefs.h
+++ b/arch/arm/plat-omap/include/dspbridge/cmmdefs.h
@@ -35,7 +35,7 @@
 #ifndef CMMDEFS_
 #define CMMDEFS_
 
-#include <list.h>
+#include <dspbridge/list.h>
 
 /* Cmm attributes used in CMM_Create() */
 	struct CMM_MGRATTRS {
--- a/arch/arm/plat-omap/include/dspbridge/cod.h
+++ b/arch/arm/plat-omap/include/dspbridge/cod.h
@@ -71,7 +71,7 @@
 #ifndef COD_
 #define COD_
 
-#include <dblldefs.h>
+#include <dspbridge/dblldefs.h>
 
 #define COD_MAXPATHLENGTH       255
 #define COD_TRACEBEG            "SYS_PUTCBEG"
--- a/arch/arm/plat-omap/include/dspbridge/csl.h
+++ b/arch/arm/plat-omap/include/dspbridge/csl.h
@@ -49,7 +49,7 @@
 #ifndef CSL_
 #define CSL_
 
-#include <host_os.h>
+#include <dspbridge/host_os.h>
 
 /*
  *  ======== CSL_Exit ========
--- a/arch/arm/plat-omap/include/dspbridge/dbc.h
+++ b/arch/arm/plat-omap/include/dspbridge/dbc.h
@@ -32,7 +32,7 @@
  *
  *! Revision History:
  *! ================
- *! 11-Aug-2000 ag: Removed include <std.h>
+ *! 11-Aug-2000 ag: Removed include <dspbridge/std.h>
  *! 22-Apr-1996 gp: Created.
  */
 
@@ -46,7 +46,7 @@
 /* Assertion Macros: */
 #if GT_TRACE
 
-#include <gt.h>
+#include <dspbridge/gt.h>
 
 #define DBC_Assert(exp) \
     if (!(exp)) \
--- a/arch/arm/plat-omap/include/dspbridge/dbdcd.h
+++ b/arch/arm/plat-omap/include/dspbridge/dbdcd.h
@@ -41,9 +41,9 @@
 #ifndef DBDCD_
 #define DBDCD_
 
-#include <dbdcddef.h>
-#include <host_os.h>
-#include <nldrdefs.h>
+#include <dspbridge/dbdcddef.h>
+#include <dspbridge/host_os.h>
+#include <dspbridge/nldrdefs.h>
 
 /*
  *  ======== DCD_AutoRegister ========
--- a/arch/arm/plat-omap/include/dspbridge/dbdcddef.h
+++ b/arch/arm/plat-omap/include/dspbridge/dbdcddef.h
@@ -42,8 +42,8 @@
 #ifndef DBDCDDEF_
 #define DBDCDDEF_
 
-#include <dbdefs.h>
-#include <mgrpriv.h>		/* for MGR_PROCESSOREXTINFO */
+#include <dspbridge/dbdefs.h>
+#include <dspbridge/mgrpriv.h>		/* for MGR_PROCESSOREXTINFO */
 
 /*
  *  The following defines are critical elements for the DCD module:
--- a/arch/arm/plat-omap/include/dspbridge/dbdefs.h
+++ b/arch/arm/plat-omap/include/dspbridge/dbdefs.h
@@ -70,9 +70,9 @@
 
 #include <linux/types.h>
 
-#include <dbtype.h>		/* GPP side type definitions           */
-#include <std.h>		/* DSP/BIOS type definitions           */
-#include <rms_sh.h>		/* Types shared between GPP and DSP    */
+#include <dspbridge/dbtype.h>		/* GPP side type definitions           */
+#include <dspbridge/std.h>		/* DSP/BIOS type definitions           */
+#include <dspbridge/rms_sh.h>		/* Types shared between GPP and DSP    */
 
 #define PG_SIZE_4K 4096
 #define PG_MASK(pg_size) (~((pg_size)-1))
--- a/arch/arm/plat-omap/include/dspbridge/dbg.h
+++ b/arch/arm/plat-omap/include/dspbridge/dbg.h
@@ -39,7 +39,7 @@
 
 #ifndef DBG_
 #define DBG_
-#include <host_os.h>
+#include <dspbridge/host_os.h>
 #include <linux/types.h>
 
 /* Levels of trace debug messages: */
--- a/arch/arm/plat-omap/include/dspbridge/dbl.h
+++ b/arch/arm/plat-omap/include/dspbridge/dbl.h
@@ -31,8 +31,8 @@
 #ifndef DBL_
 #define DBL_
 
-#include <dbdefs.h>
-#include <dbldefs.h>
+#include <dspbridge/dbdefs.h>
+#include <dspbridge/dbldefs.h>
 
 /*
  *  ======== DBL_close ========
--- a/arch/arm/plat-omap/include/dspbridge/dbll.h
+++ b/arch/arm/plat-omap/include/dspbridge/dbll.h
@@ -29,8 +29,8 @@
 #ifndef DBLL_
 #define DBLL_
 
-#include <dbdefs.h>
-#include <dblldefs.h>
+#include <dspbridge/dbdefs.h>
+#include <dspbridge/dblldefs.h>
 
 	extern void DBLL_close(struct DBLL_LibraryObj *lib);
 	extern DSP_STATUS DBLL_create(struct DBLL_TarObj **pTarget,
--- a/arch/arm/plat-omap/include/dspbridge/dehdefs.h
+++ b/arch/arm/plat-omap/include/dspbridge/dehdefs.h
@@ -21,7 +21,7 @@
  *
  *! Revision History:
  *! ================
- *! 17-Dec-2001 ag: added #include <mbx_sh.h> for shared mailbox codes.
+ *! 17-Dec-2001 ag: added #include <dspbridge/mbx_sh.h> for shared mailbox codes.
  *! 10-Dec-2001 kc: added DEH error base value and error max value.
  *! 11-Sep-2001 kc: created.
  */
@@ -29,7 +29,7 @@
 #ifndef DEHDEFS_
 #define DEHDEFS_
 
-#include <mbx_sh.h>		/* shared mailbox codes */
+#include <dspbridge/mbx_sh.h>		/* shared mailbox codes */
 
 /* DEH object manager */
 	struct DEH_MGR;
--- a/arch/arm/plat-omap/include/dspbridge/dev.h
+++ b/arch/arm/plat-omap/include/dspbridge/dev.h
@@ -80,18 +80,18 @@
 #define DEV_
 
 /*  ----------------------------------- Module Dependent Headers */
-#include <chnldefs.h>
-#include <cmm.h>
-#include <cod.h>
-#include <dehdefs.h>
-#include <nodedefs.h>
-#include <dispdefs.h>
-#include <wmd.h>
-#include <dmm.h>
-#include <host_os.h>
+#include <dspbridge/chnldefs.h>
+#include <dspbridge/cmm.h>
+#include <dspbridge/cod.h>
+#include <dspbridge/dehdefs.h>
+#include <dspbridge/nodedefs.h>
+#include <dspbridge/dispdefs.h>
+#include <dspbridge/wmd.h>
+#include <dspbridge/dmm.h>
+#include <dspbridge/host_os.h>
 
 /*  ----------------------------------- This */
-#include <devdefs.h>
+#include <dspbridge/devdefs.h>
 
 
 /*
--- a/arch/arm/plat-omap/include/dspbridge/disp.h
+++ b/arch/arm/plat-omap/include/dspbridge/disp.h
@@ -46,10 +46,10 @@
 #ifndef DISP_
 #define DISP_
 
-#include <dbdefs.h>
-#include <nodedefs.h>
-#include <nodepriv.h>
-#include <dispdefs.h>
+#include <dspbridge/dbdefs.h>
+#include <dspbridge/nodedefs.h>
+#include <dspbridge/nodepriv.h>
+#include <dspbridge/dispdefs.h>
 
 /*
  *  ======== DISP_Create ========
--- a/arch/arm/plat-omap/include/dspbridge/dmm.h
+++ b/arch/arm/plat-omap/include/dspbridge/dmm.h
@@ -32,7 +32,7 @@
 #ifndef DMM_
 #define DMM_
 
-#include <dbdefs.h>
+#include <dspbridge/dbdefs.h>
 
 	struct DMM_OBJECT;
 
--- a/arch/arm/plat-omap/include/dspbridge/drv.h
+++ b/arch/arm/plat-omap/include/dspbridge/drv.h
@@ -61,9 +61,9 @@
 #ifndef DRV_
 #define DRV_
 
-#include <devdefs.h>
+#include <dspbridge/devdefs.h>
 
-#include <drvdefs.h>
+#include <dspbridge/drvdefs.h>
 
 #define DRV_ASSIGN     1
 #define DRV_RELEASE    0
--- a/arch/arm/plat-omap/include/dspbridge/gb.h
+++ b/arch/arm/plat-omap/include/dspbridge/gb.h
@@ -26,7 +26,7 @@
 #define GB_
 
 #define GB_NOBITS (~0)
-#include <host_os.h>
+#include <dspbridge/host_os.h>
 typedef s32 GB_BitNum;
 struct GB_TMap;
 
--- a/arch/arm/plat-omap/include/dspbridge/gh.h
+++ b/arch/arm/plat-omap/include/dspbridge/gh.h
@@ -24,7 +24,7 @@
 
 #ifndef GH_
 #define GH_
-#include <host_os.h>
+#include <dspbridge/host_os.h>
 
 extern struct GH_THashTab *GH_create(u16 maxBucket, u16 valSize,
 		u16(*hash) (void *, u16), bool(*match) (void *, void *),
--- a/arch/arm/plat-omap/include/dspbridge/gt.h
+++ b/arch/arm/plat-omap/include/dspbridge/gt.h
@@ -41,7 +41,7 @@
 #define GT_TRACE 0	    /* 0 = "trace compiled out"; 1 = "trace active" */
 #endif
 
-#include <host_os.h>
+#include <dspbridge/host_os.h>
 
 #if !defined(GT_ASSERT) || GT_TRACE
 #define GT_ASSERT 1
--- a/arch/arm/plat-omap/include/dspbridge/host_os.h
+++ b/arch/arm/plat-omap/include/dspbridge/host_os.h
@@ -53,7 +53,7 @@
 #include <linux/vmalloc.h>
 #include <linux/ioport.h>
 #include <linux/platform_device.h>
-#include <dbtype.h>
+#include <dspbridge/dbtype.h>
 
 #if defined(CONFIG_ARCH_OMAP2430) || defined(CONFIG_ARCH_OMAP3430)
 #include <mach/clock.h>
--- a/arch/arm/plat-omap/include/dspbridge/io.h
+++ b/arch/arm/plat-omap/include/dspbridge/io.h
@@ -36,10 +36,10 @@
 #ifndef IO_
 #define IO_
 
-#include <cfgdefs.h>
-#include <devdefs.h>
+#include <dspbridge/cfgdefs.h>
+#include <dspbridge/devdefs.h>
 
-#include <iodefs.h>
+#include <dspbridge/iodefs.h>
 
 /*
  *  ======== IO_Create ========
--- a/arch/arm/plat-omap/include/dspbridge/io_sm.h
+++ b/arch/arm/plat-omap/include/dspbridge/io_sm.h
@@ -52,10 +52,10 @@
 #ifndef IOSM_
 #define IOSM_
 
-#include <_chnl_sm.h>
-#include <host_os.h>
+#include <dspbridge/_chnl_sm.h>
+#include <dspbridge/host_os.h>
 
-#include <iodefs.h>
+#include <dspbridge/iodefs.h>
 
 #define IO_INPUT            0
 #define IO_OUTPUT           1
--- a/arch/arm/plat-omap/include/dspbridge/list.h
+++ b/arch/arm/plat-omap/include/dspbridge/list.h
@@ -50,7 +50,7 @@
 #ifndef LIST_
 #define LIST_
 
-#include <host_os.h>
+#include <dspbridge/host_os.h>
 
 #define LST_IsEmpty(l)      (((l)->head.next == &(l)->head))
 
--- a/arch/arm/plat-omap/include/dspbridge/mem.h
+++ b/arch/arm/plat-omap/include/dspbridge/mem.h
@@ -68,8 +68,8 @@
 #ifndef MEM_
 #define MEM_
 
-#include <host_os.h>
-#include <memdefs.h>
+#include <dspbridge/host_os.h>
+#include <dspbridge/memdefs.h>
 
 /*
  *  ======== MEM_Alloc ========
--- a/arch/arm/plat-omap/include/dspbridge/mgr.h
+++ b/arch/arm/plat-omap/include/dspbridge/mgr.h
@@ -47,7 +47,7 @@
 #ifndef MGR_
 #define MGR_
 
-#include <mgrpriv.h>
+#include <dspbridge/mgrpriv.h>
 
 #define MAX_EVENTS 32
 
--- a/arch/arm/plat-omap/include/dspbridge/msg.h
+++ b/arch/arm/plat-omap/include/dspbridge/msg.h
@@ -39,8 +39,8 @@
 #ifndef MSG_
 #define MSG_
 
-#include <devdefs.h>
-#include <msgdefs.h>
+#include <dspbridge/devdefs.h>
+#include <dspbridge/msgdefs.h>
 
 /*
  *  ======== MSG_Create ========
--- a/arch/arm/plat-omap/include/dspbridge/nldr.h
+++ b/arch/arm/plat-omap/include/dspbridge/nldr.h
@@ -41,11 +41,11 @@
  *! 17-Apr-2002 jeh     Created.
  */
 
-#include <dbdefs.h>
-#include <dbdcddef.h>
-#include <dev.h>
-#include <rmm.h>
-#include <nldrdefs.h>
+#include <dspbridge/dbdefs.h>
+#include <dspbridge/dbdcddef.h>
+#include <dspbridge/dev.h>
+#include <dspbridge/rmm.h>
+#include <dspbridge/nldrdefs.h>
 
 #ifndef NLDR_
 #define NLDR_
--- a/arch/arm/plat-omap/include/dspbridge/nldrdefs.h
+++ b/arch/arm/plat-omap/include/dspbridge/nldrdefs.h
@@ -29,8 +29,8 @@
 #ifndef NLDRDEFS_
 #define NLDRDEFS_
 
-#include <dbdcddef.h>
-#include <devdefs.h>
+#include <dspbridge/dbdcddef.h>
+#include <dspbridge/devdefs.h>
 
 #define NLDR_MAXPATHLENGTH       255
 /* NLDR Objects: */
--- a/arch/arm/plat-omap/include/dspbridge/node.h
+++ b/arch/arm/plat-omap/include/dspbridge/node.h
@@ -63,11 +63,11 @@
 #ifndef NODE_
 #define NODE_
 
-#include <procpriv.h>
+#include <dspbridge/procpriv.h>
 
-#include <nodedefs.h>
-#include <dispdefs.h>
-#include <nldrdefs.h>
+#include <dspbridge/nodedefs.h>
+#include <dspbridge/dispdefs.h>
+#include <dspbridge/nldrdefs.h>
 
 /*
  *  ======== NODE_Allocate ========
--- a/arch/arm/plat-omap/include/dspbridge/nodepriv.h
+++ b/arch/arm/plat-omap/include/dspbridge/nodepriv.h
@@ -38,9 +38,9 @@
 #ifndef NODEPRIV_
 #define NODEPRIV_
 
-#include <strmdefs.h>
-#include <nodedefs.h>
-#include <nldrdefs.h>
+#include <dspbridge/strmdefs.h>
+#include <dspbridge/nodedefs.h>
+#include <dspbridge/nldrdefs.h>
 
 /* DSP address of node environment structure */
 	typedef u32 NODE_ENV;
--- a/arch/arm/plat-omap/include/dspbridge/proc.h
+++ b/arch/arm/plat-omap/include/dspbridge/proc.h
@@ -62,8 +62,8 @@
 #ifndef PROC_
 #define PROC_
 
-#include <cfgdefs.h>
-#include <devdefs.h>
+#include <dspbridge/cfgdefs.h>
+#include <dspbridge/devdefs.h>
 
 /*
  *  ======== PROC_Attach ========
--- a/arch/arm/plat-omap/include/dspbridge/pwr.h
+++ b/arch/arm/plat-omap/include/dspbridge/pwr.h
@@ -36,9 +36,9 @@
 #ifndef PWR_
 #define PWR_
 
-#include <dbdefs.h>
-#include <errbase.h>
-#include <pwr_sh.h>
+#include <dspbridge/dbdefs.h>
+#include <dspbridge/errbase.h>
+#include <dspbridge/pwr_sh.h>
 
 /*
  *  ======== PWR_SleepDSP ========
--- a/arch/arm/plat-omap/include/dspbridge/pwr_sh.h
+++ b/arch/arm/plat-omap/include/dspbridge/pwr_sh.h
@@ -27,7 +27,7 @@
 #ifndef PWR_SH_
 #define PWR_SH_
 
-#include <mbx_sh.h>
+#include <dspbridge/mbx_sh.h>
 
 /* valid sleep command codes that can be sent by GPP via mailbox: */
 #define PWR_DEEPSLEEP           MBX_PM_DSPIDLE
--- a/arch/arm/plat-omap/include/dspbridge/resourcecleanup.h
+++ b/arch/arm/plat-omap/include/dspbridge/resourcecleanup.h
@@ -16,8 +16,8 @@
 
 #ifndef RES_CLEANUP_DISABLE
 
-#include <nodepriv.h>
-#include <drv.h>
+#include <dspbridge/nodepriv.h>
+#include <dspbridge/drv.h>
 
 
 extern DSP_STATUS DRV_GetProcCtxtList(struct PROCESS_CONTEXT **pPctxt,
--- a/arch/arm/plat-omap/include/dspbridge/rms_sh.h
+++ b/arch/arm/plat-omap/include/dspbridge/rms_sh.h
@@ -48,7 +48,7 @@
 #ifndef RMS_SH_
 #define RMS_SH_
 
-#include <rmstypes.h>
+#include <dspbridge/rmstypes.h>
 
 /* Node Types: */
 #define RMS_TASK                1	/* Task node */
--- a/arch/arm/plat-omap/include/dspbridge/rmstypes.h
+++ b/arch/arm/plat-omap/include/dspbridge/rmstypes.h
@@ -33,7 +33,7 @@
 /*
  *  DSP-side definitions.
  */
-#include <std.h>
+#include <dspbridge/std.h>
 typedef u32 RMS_WORD;
 typedef char RMS_CHAR;
 
--- a/arch/arm/plat-omap/include/dspbridge/services.h
+++ b/arch/arm/plat-omap/include/dspbridge/services.h
@@ -32,7 +32,7 @@
 #ifndef SERVICES_
 #define SERVICES_
 
-#include <host_os.h>
+#include <dspbridge/host_os.h>
 /*
  *  ======== SERVICES_Exit ========
  *  Purpose:
--- a/arch/arm/plat-omap/include/dspbridge/std.h
+++ b/arch/arm/plat-omap/include/dspbridge/std.h
@@ -38,7 +38,7 @@
  *! 11-Jul-1997 dlr:	_5t_, and STD_SPOXTASK keyword for Tasking
  *! 12-Jun-1997 mf: 	_TMS320C60 -> _TMS320C6200
  *! 13-Feb-1997 mf:	_62_, with 32-bit LgInt
- *! 26-Nov-1996 kw: 	merged bios-c00's and wsx-a27's <std.h> changes
+ *! 26-Nov-1996 kw: 	merged bios-c00's and wsx-a27's <dspbridge/std.h> changes
  *!			*and* revision history
  *! 12-Sep-1996 kw: 	added C54x #ifdef's
  *! 21-Aug-1996 mf: 	removed #define main smain for _21_
--- a/arch/arm/plat-omap/include/dspbridge/strm.h
+++ b/arch/arm/plat-omap/include/dspbridge/strm.h
@@ -56,9 +56,9 @@
 #ifndef STRM_
 #define STRM_
 
-#include <dev.h>
+#include <dspbridge/dev.h>
 
-#include <strmdefs.h>
+#include <dspbridge/strmdefs.h>
 
 /*
  *  ======== STRM_AllocateBuffer ========
--- a/arch/arm/plat-omap/include/dspbridge/util.h
+++ b/arch/arm/plat-omap/include/dspbridge/util.h
@@ -53,7 +53,7 @@
 #include <linux/delay.h>
 #include <linux/sched.h>
 
-#include <utildefs.h>
+#include <dspbridge/utildefs.h>
 
 /*
  *  ======== UTIL_CDTestDll ========
--- a/arch/arm/plat-omap/include/dspbridge/wcdioctl.h
+++ b/arch/arm/plat-omap/include/dspbridge/wcdioctl.h
@@ -66,10 +66,10 @@
 #ifndef WCDIOCTL_
 #define WCDIOCTL_
 
-#include <mem.h>
-#include <cmm.h>
-#include <strmdefs.h>
-#include <dbdcd.h>
+#include <dspbridge/mem.h>
+#include <dspbridge/cmm.h>
+#include <dspbridge/strmdefs.h>
+#include <dspbridge/dbdcd.h>
 
 union Trapped_Args {
 
--- a/arch/arm/plat-omap/include/dspbridge/wmd.h
+++ b/arch/arm/plat-omap/include/dspbridge/wmd.h
@@ -62,13 +62,13 @@
 #ifndef WMD_
 #define WMD_
 
-#include <brddefs.h>
-#include <cfgdefs.h>
-#include <chnlpriv.h>
-#include <dehdefs.h>
-#include <devdefs.h>
-#include <iodefs.h>
-#include <msgdefs.h>
+#include <dspbridge/brddefs.h>
+#include <dspbridge/cfgdefs.h>
+#include <dspbridge/chnlpriv.h>
+#include <dspbridge/dehdefs.h>
+#include <dspbridge/devdefs.h>
+#include <dspbridge/iodefs.h>
+#include <dspbridge/msgdefs.h>
 
 /*
  *  Any IOCTLS at or above this value are reserved for standard WMD
--- a/arch/arm/plat-omap/include/dspbridge/wmddeh.h
+++ b/arch/arm/plat-omap/include/dspbridge/wmddeh.h
@@ -41,9 +41,9 @@
 #ifndef WMDDEH_
 #define WMDDEH_
 
-#include <devdefs.h>
+#include <dspbridge/devdefs.h>
 
-#include <dehdefs.h>
+#include <dspbridge/dehdefs.h>
 
 	extern DSP_STATUS WMD_DEH_Create(OUT struct DEH_MGR **phDehMgr,
 					 struct DEV_OBJECT *hDevObject);
--- a/arch/arm/plat-omap/include/dspbridge/wmdio.h
+++ b/arch/arm/plat-omap/include/dspbridge/wmdio.h
@@ -35,8 +35,8 @@
 #ifndef WMDIO_
 #define WMDIO_
 
-#include <devdefs.h>
-#include <iodefs.h>
+#include <dspbridge/devdefs.h>
+#include <dspbridge/iodefs.h>
 
 	extern DSP_STATUS WMD_IO_Create(OUT struct IO_MGR **phIOMgr,
 					struct DEV_OBJECT *hDevObject,
--- a/arch/arm/plat-omap/include/dspbridge/wmdmsg.h
+++ b/arch/arm/plat-omap/include/dspbridge/wmdmsg.h
@@ -37,7 +37,7 @@
 #ifndef WMDMSG_
 #define WMDMSG_
 
-#include <msgdefs.h>
+#include <dspbridge/msgdefs.h>
 
 	extern DSP_STATUS WMD_MSG_Create(OUT struct MSG_MGR **phMsgMgr,
 					 struct DEV_OBJECT *hDevObject,
--- a/drivers/dsp/bridge/Kbuild
+++ b/drivers/dsp/bridge/Kbuild
@@ -37,6 +37,4 @@ ccflags-y += -Idrivers/dsp/bridge/servic
 ccflags-y += -Idrivers/dsp/bridge/wmd
 ccflags-y += -Idrivers/dsp/bridge/pmgr
 ccflags-y += -Idrivers/dsp/bridge/rmgr
-
-ccflags-y += -Iarch/arm/plat-omap/include/dspbridge
 ccflags-y += -Idrivers/dsp/bridge/hw
\ No newline at end of file
--- a/drivers/dsp/bridge/dynload/dlclasses_hdr.h
+++ b/drivers/dsp/bridge/dynload/dlclasses_hdr.h
@@ -31,7 +31,7 @@
  *****************************************************************************
  *****************************************************************************/
 
-#include "dynamic_loader.h"
+#include <dspbridge/dynamic_loader.h>
 
 #include "DLstream.h"
 #include "DLsymtab.h"
--- a/drivers/dsp/bridge/dynload/getsection.c
+++ b/drivers/dsp/bridge/dynload/getsection.c
@@ -16,7 +16,7 @@
 
 
 
-#include "getsection.h"
+#include <dspbridge/getsection.h>
 #include "header.h"
 
 /*
--- a/drivers/dsp/bridge/dynload/header.h
+++ b/drivers/dsp/bridge/dynload/header.h
@@ -33,7 +33,7 @@ typedef unsigned int            uint_lea
 typedef unsigned short int	uint_least16_t;
 
 #include "doff.h"
-#include "dynamic_loader.h"
+#include <dspbridge/dynamic_loader.h>
 #include "params.h"
 #include "dload_internal.h"
 #include "reloc_table.h"
--- a/drivers/dsp/bridge/gen/_gt_para.c
+++ b/drivers/dsp/bridge/gen/_gt_para.c
@@ -35,13 +35,13 @@
  */
 
 /*  ----------------------------------- Host OS */
-#include <host_os.h>
+#include <dspbridge/host_os.h>
 
 /*  ----------------------------------- DSP/BIOS Bridge */
-#include <std.h>
+#include <dspbridge/std.h>
 
 /*  ----------------------------------- Trace & Debug */
-#include <gt.h>
+#include <dspbridge/gt.h>
 
 /*  ----------------------------------- Function Prototypes */
 static void error(char *msg, ...);
--- a/drivers/dsp/bridge/gen/gb.c
+++ b/drivers/dsp/bridge/gen/gb.c
@@ -30,11 +30,11 @@
  */
 
 /*  ----------------------------------- DSP/BIOS Bridge */
-#include <std.h>
+#include <dspbridge/std.h>
 #include <linux/types.h>
 /*  ----------------------------------- This */
-#include <gs.h>
-#include <gb.h>
+#include <dspbridge/gs.h>
+#include <dspbridge/gb.h>
 
 typedef GB_BitNum GB_WordNum;
 
--- a/drivers/dsp/bridge/gen/gh.c
+++ b/drivers/dsp/bridge/gen/gh.c
@@ -19,13 +19,13 @@
  *  ======== gh.c ========
  */
 
-#include <std.h>
+#include <dspbridge/std.h>
 
-#include <host_os.h>
+#include <dspbridge/host_os.h>
 
-#include <gs.h>
+#include <dspbridge/gs.h>
 
-#include <gh.h>
+#include <dspbridge/gh.h>
 
 struct Elem {
 	struct Elem *next;
--- a/drivers/dsp/bridge/gen/gs.c
+++ b/drivers/dsp/bridge/gen/gs.c
@@ -30,14 +30,14 @@
  */
 
 /*  ----------------------------------- DSP/BIOS Bridge */
-#include <std.h>
-#include <dbdefs.h>
+#include <dspbridge/std.h>
+#include <dspbridge/dbdefs.h>
 #include <linux/types.h>
 /*  ----------------------------------- OS Adaptation Layer */
-#include <mem.h>
+#include <dspbridge/mem.h>
 
 /*  ----------------------------------- This */
-#include <gs.h>
+#include <dspbridge/gs.h>
 
 /*  ----------------------------------- Globals */
 static u32 cumsize;
--- a/drivers/dsp/bridge/gen/gt.c
+++ b/drivers/dsp/bridge/gen/gt.c
@@ -29,10 +29,10 @@
  */
 
 /*  ----------------------------------- DSP/BIOS Bridge */
-#include <std.h>
+#include <dspbridge/std.h>
 
 /*  ----------------------------------- This */
-#include <gt.h>
+#include <dspbridge/gt.h>
 
 #define GT_WILD	'*'
 
--- a/drivers/dsp/bridge/gen/uuidutil.c
+++ b/drivers/dsp/bridge/gen/uuidutil.c
@@ -33,17 +33,17 @@
  */
 
 /*  ----------------------------------- Host OS  */
-#include <host_os.h>
+#include <dspbridge/host_os.h>
 
 /*  ----------------------------------- DSP/BIOS Bridge */
-#include <std.h>
-#include <dbdefs.h>
+#include <dspbridge/std.h>
+#include <dspbridge/dbdefs.h>
 
 /*  ----------------------------------- Trace & Debug */
-#include <dbc.h>
+#include <dspbridge/dbc.h>
 
 /*  ----------------------------------- This */
-#include <uuidutil.h>
+#include <dspbridge/uuidutil.h>
 
 /*
  *  ======== UUID_UuidToString ========
--- a/drivers/dsp/bridge/pmgr/chnl.c
+++ b/drivers/dsp/bridge/pmgr/chnl.c
@@ -63,35 +63,35 @@
  */
 
 /*  ----------------------------------- Host OS */
-#include <host_os.h>
+#include <dspbridge/host_os.h>
 
 /*  ----------------------------------- DSP/BIOS Bridge */
-#include <std.h>
-#include <dbdefs.h>
-#include <errbase.h>
+#include <dspbridge/std.h>
+#include <dspbridge/dbdefs.h>
+#include <dspbridge/errbase.h>
 
 /*  ----------------------------------- Trace & Debug */
-#include <dbc.h>
-#include <gt.h>
+#include <dspbridge/dbc.h>
+#include <dspbridge/gt.h>
 
 /*  ----------------------------------- OS Adaptation Layer */
-#include <cfg.h>
-#include <csl.h>
-#include <dpc.h>
-#include <list.h>
-#include <mem.h>
-#include <sync.h>
+#include <dspbridge/cfg.h>
+#include <dspbridge/csl.h>
+#include <dspbridge/dpc.h>
+#include <dspbridge/list.h>
+#include <dspbridge/mem.h>
+#include <dspbridge/sync.h>
 
 /*  ----------------------------------- Platform Manager */
-#include <proc.h>
-#include <dev.h>
+#include <dspbridge/proc.h>
+#include <dspbridge/dev.h>
 
 /*  ----------------------------------- Others */
-#include <chnlpriv.h>
+#include <dspbridge/chnlpriv.h>
 #include <chnlobj.h>
 
 /*  ----------------------------------- This */
-#include <chnl.h>
+#include <dspbridge/chnl.h>
 
 /*  ----------------------------------- Globals */
 static u32 cRefs;
--- a/drivers/dsp/bridge/pmgr/chnlobj.h
+++ b/drivers/dsp/bridge/pmgr/chnlobj.h
@@ -35,8 +35,8 @@
 #ifndef CHNLOBJ_
 #define CHNLOBJ_
 
-#include <chnldefs.h>
-#include <wmd.h>
+#include <dspbridge/chnldefs.h>
+#include <dspbridge/wmd.h>
 
 /* Object validateion macros: */
 #define CHNL_IsValidMgr(h) \
--- a/drivers/dsp/bridge/pmgr/cmm.c
+++ b/drivers/dsp/bridge/pmgr/cmm.c
@@ -94,27 +94,27 @@
  */
 
 /*  ----------------------------------- DSP/BIOS Bridge */
-#include <std.h>
-#include <dbdefs.h>
+#include <dspbridge/std.h>
+#include <dspbridge/dbdefs.h>
 
 /*  ----------------------------------- Trace & Debug */
-#include <dbc.h>
-#include <errbase.h>
-#include <gt.h>
+#include <dspbridge/dbc.h>
+#include <dspbridge/errbase.h>
+#include <dspbridge/gt.h>
 
 /*  ----------------------------------- OS Adaptation Layer */
-#include <cfg.h>
-#include <list.h>
-#include <mem.h>
-#include <sync.h>
-#include <util.h>
+#include <dspbridge/cfg.h>
+#include <dspbridge/list.h>
+#include <dspbridge/mem.h>
+#include <dspbridge/sync.h>
+#include <dspbridge/util.h>
 
 /*  ----------------------------------- Platform Manager */
-#include <dev.h>
-#include <proc.h>
+#include <dspbridge/dev.h>
+#include <dspbridge/proc.h>
 
 /*  ----------------------------------- This */
-#include <cmm.h>
+#include <dspbridge/cmm.h>
 
 /*  ----------------------------------- Defines, Data Structures, Typedefs */
 /* Object signatures */
--- a/drivers/dsp/bridge/pmgr/cod.c
+++ b/drivers/dsp/bridge/pmgr/cod.c
@@ -56,29 +56,29 @@
  */
 
 /*  ----------------------------------- Host OS */
-#include <host_os.h>
+#include <dspbridge/host_os.h>
 
 /*  ----------------------------------- DSP/BIOS Bridge */
-#include <std.h>
-#include <dbdefs.h>
-#include <errbase.h>
+#include <dspbridge/std.h>
+#include <dspbridge/dbdefs.h>
+#include <dspbridge/errbase.h>
 
 /*  ----------------------------------- Trace & Debug */
-#include <dbc.h>
-#include <gt.h>
+#include <dspbridge/dbc.h>
+#include <dspbridge/gt.h>
 
 /*  ----------------------------------- OS Adaptation Layer */
-#include <csl.h>
-#include <kfile.h>
-#include <ldr.h>
-#include <mem.h>
+#include <dspbridge/csl.h>
+#include <dspbridge/kfile.h>
+#include <dspbridge/ldr.h>
+#include <dspbridge/mem.h>
 
 /*  ----------------------------------- Platform Manager */
 /* Include appropriate loader header file */
-#include <dbll.h>
+#include <dspbridge/dbll.h>
 
 /*  ----------------------------------- This */
-#include <cod.h>
+#include <dspbridge/cod.h>
 
 /* magic number for handle validation */
 #define MAGIC	 0xc001beef
--- a/drivers/dsp/bridge/pmgr/dbl.c
+++ b/drivers/dsp/bridge/pmgr/dbl.c
@@ -36,25 +36,25 @@
  */
 
 /*  ----------------------------------- Host OS */
-#include <host_os.h>
+#include <dspbridge/host_os.h>
 
 /*  ----------------------------------- DSP/BIOS Bridge */
-#include <std.h>
-#include <dbdefs.h>
-#include <errbase.h>
+#include <dspbridge/std.h>
+#include <dspbridge/dbdefs.h>
+#include <dspbridge/errbase.h>
 
 /*  ----------------------------------- Trace & Debug */
-#include <dbc.h>
-#include <gt.h>
+#include <dspbridge/dbc.h>
+#include <dspbridge/gt.h>
 
 /*  ----------------------------------- OS Adaptation Layer */
-#include <csl.h>
-#include <mem.h>
-#include <kfile.h>
+#include <dspbridge/csl.h>
+#include <dspbridge/mem.h>
+#include <dspbridge/kfile.h>
 
 /*  ----------------------------------- This */
-#include <dbof.h>
-#include <dbl.h>
+#include <dspbridge/dbof.h>
+#include <dspbridge/dbl.h>
 
 #define DBL_TARGSIGNATURE      0x544c4244	/* "TLBD" */
 #define DBL_LIBSIGNATURE       0x4c4c4244	/* "LLBD" */
--- a/drivers/dsp/bridge/pmgr/dbll.c
+++ b/drivers/dsp/bridge/pmgr/dbll.c
@@ -35,29 +35,29 @@
  */
 
 /*  ----------------------------------- Host OS */
-#include <host_os.h>
+#include <dspbridge/host_os.h>
 
 /*  ----------------------------------- DSP/BIOS Bridge */
-#include <std.h>
-#include <dbdefs.h>
-#include <errbase.h>
+#include <dspbridge/std.h>
+#include <dspbridge/dbdefs.h>
+#include <dspbridge/errbase.h>
 
 /*  ----------------------------------- Trace & Debug */
-#include <gt.h>
-#include <dbc.h>
-#include <gh.h>
+#include <dspbridge/gt.h>
+#include <dspbridge/dbc.h>
+#include <dspbridge/gh.h>
 
 /*  ----------------------------------- OS Adaptation Layer */
-#include <csl.h>
-#include <mem.h>
+#include <dspbridge/csl.h>
+#include <dspbridge/mem.h>
 
 /* Dynamic loader library interface */
-#include <dynamic_loader.h>
-#include <getsection.h>
+#include <dspbridge/dynamic_loader.h>
+#include <dspbridge/getsection.h>
 
 /*  ----------------------------------- This */
-#include <dbll.h>
-#include <rmm.h>
+#include <dspbridge/dbll.h>
+#include <dspbridge/rmm.h>
 
 #define DBLL_TARGSIGNATURE      0x544c4c44	/* "TLLD" */
 #define DBLL_LIBSIGNATURE       0x4c4c4c44	/* "LLLD" */
--- a/drivers/dsp/bridge/pmgr/dev.c
+++ b/drivers/dsp/bridge/pmgr/dev.c
@@ -84,7 +84,7 @@
  *! 06-Oct-2000 rr: DEV_Destroy2 and DEV_Create2 added.
  *! 02-Oct-2000 rr: DEV_GetNodeManager added.
  *! 11-Aug-2000 ag: Added DEV_GetCmmMgr(), CMM_Init() & CMM_Exit().
- *!                 Removed <std.h> & <stdwin.h>, added <dbtype.h>
+ *!                 Removed <dspbridge/std.h> & <stdwin.h>, added <dspbridge/dbtype.h>
  *! 10-Aug-2000 rr: DEV_InsertProcObject/RemoveProcObject added.
  *!                 DEV_Cleanup calls PROC_Detach if it is a matching process.
  *! 27-Jul-2000 rr: DEV is in new directoy DEV and produces devlib.lib
@@ -113,46 +113,46 @@
  */
 
 /*  ----------------------------------- Host OS */
-#include <host_os.h>
+#include <dspbridge/host_os.h>
 
 /*  ----------------------------------- DSP/BIOS Bridge */
-#include <std.h>
-#include <dbdefs.h>
-#include <errbase.h>
+#include <dspbridge/std.h>
+#include <dspbridge/dbdefs.h>
+#include <dspbridge/errbase.h>
 
 /*  ----------------------------------- Trace & Debug */
-#include <dbc.h>
-#include <dbg.h>
-#include <gt.h>
+#include <dspbridge/dbc.h>
+#include <dspbridge/dbg.h>
+#include <dspbridge/gt.h>
 
 /*  ----------------------------------- OS Adaptation Layer */
-#include <cfg.h>
-#include <ldr.h>
-#include <list.h>
-#include <mem.h>
-#include <util.h>
+#include <dspbridge/cfg.h>
+#include <dspbridge/ldr.h>
+#include <dspbridge/list.h>
+#include <dspbridge/mem.h>
+#include <dspbridge/util.h>
 
 /*  ----------------------------------- Platform Manager */
-#include <cod.h>
-#include <drv.h>
-#include <proc.h>
-#include <dmm.h>
+#include <dspbridge/cod.h>
+#include <dspbridge/drv.h>
+#include <dspbridge/proc.h>
+#include <dspbridge/dmm.h>
 
 /*  ----------------------------------- Resource Manager */
-#include <mgr.h>
-#include <node.h>
+#include <dspbridge/mgr.h>
+#include <dspbridge/node.h>
 
 /*  ----------------------------------- Others */
-#include <dbreg.h>
-#include <wcd.h>		/* WCD version info. */
+#include <dspbridge/dbreg.h>
+#include <dspbridge/wcd.h>		/* WCD version info. */
 
-#include <chnl.h>
-#include <io.h>
-#include <msg.h>
-#include <cmm.h>
+#include <dspbridge/chnl.h>
+#include <dspbridge/io.h>
+#include <dspbridge/msg.h>
+#include <dspbridge/cmm.h>
 
 /*  ----------------------------------- This */
-#include <dev.h>
+#include <dspbridge/dev.h>
 
 /*  ----------------------------------- Defines, Data Structures, Typedefs */
 
--- a/drivers/dsp/bridge/pmgr/dmm.c
+++ b/drivers/dsp/bridge/pmgr/dmm.c
@@ -55,28 +55,28 @@
  */
 
 /*  ----------------------------------- Host OS */
-#include <host_os.h>
+#include <dspbridge/host_os.h>
 
 /*  ----------------------------------- DSP/BIOS Bridge */
-#include <std.h>
-#include <dbdefs.h>
+#include <dspbridge/std.h>
+#include <dspbridge/dbdefs.h>
 
 /*  ----------------------------------- Trace & Debug */
-#include <dbc.h>
-#include <errbase.h>
-#include <gt.h>
+#include <dspbridge/dbc.h>
+#include <dspbridge/errbase.h>
+#include <dspbridge/gt.h>
 
 /*  ----------------------------------- OS Adaptation Layer */
-#include <list.h>
-#include <mem.h>
-#include <sync.h>
+#include <dspbridge/list.h>
+#include <dspbridge/mem.h>
+#include <dspbridge/sync.h>
 
 /*  ----------------------------------- Platform Manager */
-#include <dev.h>
-#include <proc.h>
+#include <dspbridge/dev.h>
+#include <dspbridge/proc.h>
 
 /*  ----------------------------------- This */
-#include <dmm.h>
+#include <dspbridge/dmm.h>
 
 /*  ----------------------------------- Defines, Data Structures, Typedefs */
 /* Object signatures */
--- a/drivers/dsp/bridge/pmgr/io.c
+++ b/drivers/dsp/bridge/pmgr/io.c
@@ -38,28 +38,28 @@
  */
 
 /*  ----------------------------------- Host OS */
-#include <host_os.h>
+#include <dspbridge/host_os.h>
 
 /*  ----------------------------------- DSP/BIOS Bridge */
-#include <std.h>
-#include <dbdefs.h>
-#include <errbase.h>
+#include <dspbridge/std.h>
+#include <dspbridge/dbdefs.h>
+#include <dspbridge/errbase.h>
 
 /*  ----------------------------------- Trace & Debug */
-#include <dbc.h>
-#include <gt.h>
+#include <dspbridge/dbc.h>
+#include <dspbridge/gt.h>
 
 /*  ----------------------------------- OS Adaptation Layer */
-#include <cfg.h>
-#include <mem.h>
+#include <dspbridge/cfg.h>
+#include <dspbridge/mem.h>
 
 /*  ----------------------------------- Platform Manager */
-#include <dev.h>
+#include <dspbridge/dev.h>
 
 /*  ----------------------------------- This */
 #include <ioobj.h>
-#include <iodefs.h>
-#include <io.h>
+#include <dspbridge/iodefs.h>
+#include <dspbridge/io.h>
 
 /*  ----------------------------------- Globals */
 static u32 cRefs;
--- a/drivers/dsp/bridge/pmgr/ioobj.h
+++ b/drivers/dsp/bridge/pmgr/ioobj.h
@@ -33,8 +33,8 @@
 #ifndef IOOBJ_
 #define IOOBJ_
 
-#include <devdefs.h>
-#include <wmd.h>
+#include <dspbridge/devdefs.h>
+#include <dspbridge/wmd.h>
 
 /*
  *  This struct is the first field in a IO_MGR struct, as implemented in
--- a/drivers/dsp/bridge/pmgr/msg.c
+++ b/drivers/dsp/bridge/pmgr/msg.c
@@ -36,30 +36,30 @@
  */
 
 /*  ----------------------------------- Host OS */
-#include <host_os.h>
+#include <dspbridge/host_os.h>
 
 /*  ----------------------------------- DSP/BIOS Bridge */
-#include <std.h>
-#include <dbdefs.h>
-#include <errbase.h>
+#include <dspbridge/std.h>
+#include <dspbridge/dbdefs.h>
+#include <dspbridge/errbase.h>
 
 /*  ----------------------------------- Trace & Debug */
-#include <dbc.h>
-#include <gt.h>
+#include <dspbridge/dbc.h>
+#include <dspbridge/gt.h>
 
 /*  ----------------------------------- OS Adaptation Layer */
-#include <list.h>
-#include <mem.h>
+#include <dspbridge/list.h>
+#include <dspbridge/mem.h>
 
 /*  ----------------------------------- Mini Driver */
-#include <wmd.h>
+#include <dspbridge/wmd.h>
 
 /*  ----------------------------------- Platform Manager */
-#include <dev.h>
+#include <dspbridge/dev.h>
 
 /*  ----------------------------------- This */
 #include <msgobj.h>
-#include <msg.h>
+#include <dspbridge/msg.h>
 
 /*  ----------------------------------- Globals */
 #if GT_TRACE
--- a/drivers/dsp/bridge/pmgr/msgobj.h
+++ b/drivers/dsp/bridge/pmgr/msgobj.h
@@ -33,9 +33,9 @@
 #ifndef MSGOBJ_
 #define MSGOBJ_
 
-#include <wmd.h>
+#include <dspbridge/wmd.h>
 
-#include <msgdefs.h>
+#include <dspbridge/msgdefs.h>
 
 /*
  *  This struct is the first field in a MSG_MGR struct, as implemented in
--- a/drivers/dsp/bridge/pmgr/wcd.c
+++ b/drivers/dsp/bridge/pmgr/wcd.c
@@ -92,50 +92,50 @@
  */
 
 /*  ----------------------------------- Host OS */
-#include <host_os.h>
+#include <dspbridge/host_os.h>
 
 /*  ----------------------------------- DSP/BIOS Bridge */
-#include <std.h>
-#include <dbdefs.h>
-#include <errbase.h>
+#include <dspbridge/std.h>
+#include <dspbridge/dbdefs.h>
+#include <dspbridge/errbase.h>
 
 /*  ----------------------------------- Trace & Debug */
-#include <dbc.h>
-#include <gt.h>
+#include <dspbridge/dbc.h>
+#include <dspbridge/gt.h>
 
 /*  ----------------------------------- OS Adaptation Layer */
-#include <cfg.h>
-#include <mem.h>
-#include <ntfy.h>
-#include <services.h>
-#include <util.h>
+#include <dspbridge/cfg.h>
+#include <dspbridge/mem.h>
+#include <dspbridge/ntfy.h>
+#include <dspbridge/services.h>
+#include <dspbridge/util.h>
 
 /*  ----------------------------------- Platform Manager */
-#include <chnl.h>
-#include <dev.h>
-#include <drv.h>
+#include <dspbridge/chnl.h>
+#include <dspbridge/dev.h>
+#include <dspbridge/drv.h>
 
-#include <proc.h>
-#include <strm.h>
+#include <dspbridge/proc.h>
+#include <dspbridge/strm.h>
 
 /*  ----------------------------------- Resource Manager */
-#include <disp.h>
-#include <mgr.h>
-#include <node.h>
-#include <rmm.h>
+#include <dspbridge/disp.h>
+#include <dspbridge/mgr.h>
+#include <dspbridge/node.h>
+#include <dspbridge/rmm.h>
 
 
 /*  ----------------------------------- Others */
-#include <msg.h>
-#include <cmm.h>
-#include <io.h>
+#include <dspbridge/msg.h>
+#include <dspbridge/cmm.h>
+#include <dspbridge/io.h>
 
 /*  ----------------------------------- This */
-#include <_dcd.h>
-#include <dbdcd.h>
+#include <dspbridge/_dcd.h>
+#include <dspbridge/dbdcd.h>
 
 #ifndef RES_CLEANUP_DISABLE
-#include <resourcecleanup.h>
+#include <dspbridge/resourcecleanup.h>
 #endif
 
 /*  ----------------------------------- Defines, Data Structures, Typedefs */
--- a/drivers/dsp/bridge/rmgr/dbdcd.c
+++ b/drivers/dsp/bridge/rmgr/dbdcd.c
@@ -62,29 +62,29 @@
  */
 
 /*  ----------------------------------- Host OS */
-#include <host_os.h>
+#include <dspbridge/host_os.h>
 
 /*  ----------------------------------- DSP/BIOS Bridge */
-#include <std.h>
-#include <dbdefs.h>
-#include <errbase.h>
+#include <dspbridge/std.h>
+#include <dspbridge/dbdefs.h>
+#include <dspbridge/errbase.h>
 /*  ----------------------------------- Trace & Debug */
-#include <dbc.h>
-#include <gt.h>
+#include <dspbridge/dbc.h>
+#include <dspbridge/gt.h>
 
 /*  ----------------------------------- OS Adaptation Layer */
-#include <csl.h>
-#include <mem.h>
-#include <reg.h>
+#include <dspbridge/csl.h>
+#include <dspbridge/mem.h>
+#include <dspbridge/reg.h>
 
 /*  ----------------------------------- Platform Manager */
-#include <cod.h>
+#include <dspbridge/cod.h>
 
 /*  ----------------------------------- Others */
-#include <uuidutil.h>
+#include <dspbridge/uuidutil.h>
 
 /*  ----------------------------------- This */
-#include <dbdcd.h>
+#include <dspbridge/dbdcd.h>
 
 /*  ----------------------------------- Global defines. */
 #define SIGNATURE       0x5f444344	/* "DCD_" (in reverse). */
--- a/drivers/dsp/bridge/rmgr/disp.c
+++ b/drivers/dsp/bridge/rmgr/disp.c
@@ -47,37 +47,37 @@
  */
 
 /*  ----------------------------------- Host OS */
-#include <host_os.h>
+#include <dspbridge/host_os.h>
 
 /*  ----------------------------------- DSP/BIOS Bridge */
-#include <std.h>
-#include <dbdefs.h>
-#include <errbase.h>
+#include <dspbridge/std.h>
+#include <dspbridge/dbdefs.h>
+#include <dspbridge/errbase.h>
 
 /*  ----------------------------------- Trace & Debug */
-#include <gt.h>
-#include <dbc.h>
+#include <dspbridge/gt.h>
+#include <dspbridge/dbc.h>
 
 /*  ----------------------------------- OS Adaptation Layer */
-#include <dev.h>
-#include <mem.h>
-#include <sync.h>
-#include <csl.h>
+#include <dspbridge/dev.h>
+#include <dspbridge/mem.h>
+#include <dspbridge/sync.h>
+#include <dspbridge/csl.h>
 
 /*  ----------------------------------- Link Driver */
-#include <wmd.h>
+#include <dspbridge/wmd.h>
 
 /*  ----------------------------------- Platform Manager */
-#include <dev.h>
-#include <chnldefs.h>
+#include <dspbridge/dev.h>
+#include <dspbridge/chnldefs.h>
 
 /*  ----------------------------------- Resource Manager */
-#include <nodedefs.h>
-#include <nodepriv.h>
-#include <rms_sh.h>
+#include <dspbridge/nodedefs.h>
+#include <dspbridge/nodepriv.h>
+#include <dspbridge/rms_sh.h>
 
 /*  ----------------------------------- This */
-#include <disp.h>
+#include <dspbridge/disp.h>
 
 #define DISP_SIGNATURE       0x50534944	/* "PSID" */
 
--- a/drivers/dsp/bridge/rmgr/drv.c
+++ b/drivers/dsp/bridge/rmgr/drv.c
@@ -91,38 +91,38 @@
  */
 
 /*  ----------------------------------- Host OS */
-#include <host_os.h>
+#include <dspbridge/host_os.h>
 
 /*  ----------------------------------- DSP/BIOS Bridge */
-#include <std.h>
-#include <dbdefs.h>
-#include <errbase.h>
+#include <dspbridge/std.h>
+#include <dspbridge/dbdefs.h>
+#include <dspbridge/errbase.h>
 
 /*  ----------------------------------- Trace & Debug */
-#include <dbc.h>
-#include <gt.h>
+#include <dspbridge/dbc.h>
+#include <dspbridge/gt.h>
 
 /*  ----------------------------------- OS Adaptation Layer */
-#include <cfg.h>
-#include <csl.h>
-#include <list.h>
-#include <mem.h>
-#include <reg.h>
+#include <dspbridge/cfg.h>
+#include <dspbridge/csl.h>
+#include <dspbridge/list.h>
+#include <dspbridge/mem.h>
+#include <dspbridge/reg.h>
 
 /*  ----------------------------------- Others */
-#include <dbreg.h>
+#include <dspbridge/dbreg.h>
 
 /*  ----------------------------------- This */
-#include <drv.h>
-#include <dev.h>
+#include <dspbridge/drv.h>
+#include <dspbridge/dev.h>
 
 #ifndef RES_CLEANUP_DISABLE
-#include <node.h>
-#include <proc.h>
-#include <strm.h>
-#include <nodepriv.h>
-#include <wmdchnl.h>
-#include <resourcecleanup.h>
+#include <dspbridge/node.h>
+#include <dspbridge/proc.h>
+#include <dspbridge/strm.h>
+#include <dspbridge/nodepriv.h>
+#include <dspbridge/wmdchnl.h>
+#include <dspbridge/resourcecleanup.h>
 #endif
 
 /*  ----------------------------------- Defines, Data Structures, Typedefs */
--- a/drivers/dsp/bridge/rmgr/drv_interface.c
+++ b/drivers/dsp/bridge/rmgr/drv_interface.c
@@ -46,7 +46,7 @@
 
 /*  ----------------------------------- Host OS */
 
-#include <host_os.h>
+#include <dspbridge/host_os.h>
 #include <linux/platform_device.h>
 #include <linux/pm.h>
 
@@ -70,42 +70,42 @@
 #endif
 
 /*  ----------------------------------- DSP/BIOS Bridge */
-#include <std.h>
-#include <dbdefs.h>
-#include <errbase.h>
+#include <dspbridge/std.h>
+#include <dspbridge/dbdefs.h>
+#include <dspbridge/errbase.h>
 
 /*  ----------------------------------- Trace & Debug */
-#include <gt.h>
-#include <dbc.h>
+#include <dspbridge/gt.h>
+#include <dspbridge/dbc.h>
 
 /*  ----------------------------------- OS Adaptation Layer */
-#include <services.h>
-#include <sync.h>
-#include <reg.h>
-#include <csl.h>
+#include <dspbridge/services.h>
+#include <dspbridge/sync.h>
+#include <dspbridge/reg.h>
+#include <dspbridge/csl.h>
 
 /*  ----------------------------------- Platform Manager */
-#include <wcdioctl.h>
-#include <_dcd.h>
-#include <dspdrv.h>
-#include <dbreg.h>
+#include <dspbridge/wcdioctl.h>
+#include <dspbridge/_dcd.h>
+#include <dspbridge/dspdrv.h>
+#include <dspbridge/dbreg.h>
 
 /*  ----------------------------------- Resource Manager */
-#include <pwr.h>
+#include <dspbridge/pwr.h>
 
 /*  ----------------------------------- This */
 #include <drv_interface.h>
 
 #ifndef RES_CLEANUP_DISABLE
-#include <cfg.h>
-#include <resourcecleanup.h>
-#include <chnl.h>
-#include <proc.h>
-#include <cfg.h>
-#include <dev.h>
-#include <drvdefs.h>
-#include <drv.h>
-#include <dbreg.h>
+#include <dspbridge/cfg.h>
+#include <dspbridge/resourcecleanup.h>
+#include <dspbridge/chnl.h>
+#include <dspbridge/proc.h>
+#include <dspbridge/cfg.h>
+#include <dspbridge/dev.h>
+#include <dspbridge/drvdefs.h>
+#include <dspbridge/drv.h>
+#include <dspbridge/dbreg.h>
 #endif
 
 #ifndef DISABLE_BRIDGE_PM
--- a/drivers/dsp/bridge/rmgr/dspdrv.c
+++ b/drivers/dsp/bridge/rmgr/dspdrv.c
@@ -92,36 +92,36 @@
  */
 
 /*  ----------------------------------- Host OS */
-#include <host_os.h>
+#include <dspbridge/host_os.h>
 
 /*  ----------------------------------- DSP/BIOS Bridge */
-#include <std.h>
-#include <dbdefs.h>
-#include <errbase.h>
+#include <dspbridge/std.h>
+#include <dspbridge/dbdefs.h>
+#include <dspbridge/errbase.h>
 
 /*  ----------------------------------- Trace & Debug */
-#include <dbc.h>
-#include <gt.h>
+#include <dspbridge/dbc.h>
+#include <dspbridge/gt.h>
 
 /*  ----------------------------------- OS Adaptation Layer */
-#include <cfg.h>
-#include <csl.h>
-#include <mem.h>
-#include <reg.h>
+#include <dspbridge/cfg.h>
+#include <dspbridge/csl.h>
+#include <dspbridge/mem.h>
+#include <dspbridge/reg.h>
 
 /*  ----------------------------------- Platform Manager */
-#include <drv.h>
-#include <dev.h>
-#include <_dcd.h>
+#include <dspbridge/drv.h>
+#include <dspbridge/dev.h>
+#include <dspbridge/_dcd.h>
 
 /*  ----------------------------------- Resource Manager */
-#include <mgr.h>
+#include <dspbridge/mgr.h>
 
 /*  ----------------------------------- Others */
-#include <dbreg.h>
+#include <dspbridge/dbreg.h>
 
 /*  ----------------------------------- This */
-#include <dspdrv.h>
+#include <dspbridge/dspdrv.h>
 
 /*  ----------------------------------- Globals */
 struct GT_Mask curTrace;
--- a/drivers/dsp/bridge/rmgr/mgr.c
+++ b/drivers/dsp/bridge/rmgr/mgr.c
@@ -41,27 +41,27 @@
  */
 
 /*  ----------------------------------- DSP/BIOS Bridge */
-#include <std.h>
-#include <dbdefs.h>
-#include <errbase.h>
+#include <dspbridge/std.h>
+#include <dspbridge/dbdefs.h>
+#include <dspbridge/errbase.h>
 
 /*  ----------------------------------- Trace & Debug */
-#include <dbc.h>
-#include <gt.h>
+#include <dspbridge/dbc.h>
+#include <dspbridge/gt.h>
 
 /*  ----------------------------------- OS Adaptation Layer */
-#include <cfg.h>
-#include <mem.h>
-#include <sync.h>
+#include <dspbridge/cfg.h>
+#include <dspbridge/mem.h>
+#include <dspbridge/sync.h>
 
 /*  ----------------------------------- Others */
-#include <dbdcd.h>
-#include <dbreg.h>
-#include <drv.h>
-#include <dev.h>
+#include <dspbridge/dbdcd.h>
+#include <dspbridge/dbreg.h>
+#include <dspbridge/drv.h>
+#include <dspbridge/dev.h>
 
 /*  ----------------------------------- This */
-#include <mgr.h>
+#include <dspbridge/mgr.h>
 
 /*  ----------------------------------- Defines, Data Structures, Typedefs */
 #define ZLDLLNAME               ""
--- a/drivers/dsp/bridge/rmgr/nldr.c
+++ b/drivers/dsp/bridge/rmgr/nldr.c
@@ -50,33 +50,33 @@
  *! 05-Aug-2002 jeh Created.
  */
 
-#include <host_os.h>
+#include <dspbridge/host_os.h>
 
-#include <std.h>
-#include <dbdefs.h>
-#include <errbase.h>
+#include <dspbridge/std.h>
+#include <dspbridge/dbdefs.h>
+#include <dspbridge/errbase.h>
 
-#include <dbc.h>
-#include <gt.h>
+#include <dspbridge/dbc.h>
+#include <dspbridge/gt.h>
 #ifdef DEBUG
-#include <dbg.h>
+#include <dspbridge/dbg.h>
 #endif
 
 /* OS adaptation layer */
-#include <csl.h>
-#include <mem.h>
+#include <dspbridge/csl.h>
+#include <dspbridge/mem.h>
 
 /* Platform manager */
-#include <cod.h>
-#include <dev.h>
+#include <dspbridge/cod.h>
+#include <dspbridge/dev.h>
 
 /* Resource manager */
-#include <dbll.h>
-#include <dbdcd.h>
-#include <rmm.h>
-#include <uuidutil.h>
+#include <dspbridge/dbll.h>
+#include <dspbridge/dbdcd.h>
+#include <dspbridge/rmm.h>
+#include <dspbridge/uuidutil.h>
 
-#include <nldr.h>
+#include <dspbridge/nldr.h>
 
 #define NLDR_SIGNATURE      0x52444c4e	/* "RDLN" */
 #define NLDR_NODESIGNATURE  0x4e444c4e	/* "NDLN" */
--- a/drivers/dsp/bridge/rmgr/node.c
+++ b/drivers/dsp/bridge/rmgr/node.c
@@ -94,62 +94,62 @@
  */
 
 /*  ----------------------------------- Host OS */
-#include <host_os.h>
+#include <dspbridge/host_os.h>
 
 /*  ----------------------------------- DSP/BIOS Bridge */
-#include <std.h>
-#include <dbdefs.h>
-#include <errbase.h>
+#include <dspbridge/std.h>
+#include <dspbridge/dbdefs.h>
+#include <dspbridge/errbase.h>
 
 /*  ----------------------------------- Trace & Debug */
-#include <dbc.h>
-#include <gt.h>
+#include <dspbridge/dbc.h>
+#include <dspbridge/gt.h>
 
 /*  ----------------------------------- OS Adaptation Layer */
-#include <cfg.h>
-#include <csl.h>
-#include <list.h>
-#include <mem.h>
-#include <proc.h>
-#include <strm.h>
-#include <sync.h>
-#include <ntfy.h>
+#include <dspbridge/cfg.h>
+#include <dspbridge/csl.h>
+#include <dspbridge/list.h>
+#include <dspbridge/mem.h>
+#include <dspbridge/proc.h>
+#include <dspbridge/strm.h>
+#include <dspbridge/sync.h>
+#include <dspbridge/ntfy.h>
 
 /*  ----------------------------------- Platform Manager */
-#include <cmm.h>
-#include <cod.h>
-#include <dev.h>
-#include <msg.h>
+#include <dspbridge/cmm.h>
+#include <dspbridge/cod.h>
+#include <dspbridge/dev.h>
+#include <dspbridge/msg.h>
 
 /*  ----------------------------------- Resource Manager */
-#include <dbdcd.h>
-#include <disp.h>
-#include <rms_sh.h>
+#include <dspbridge/dbdcd.h>
+#include <dspbridge/disp.h>
+#include <dspbridge/rms_sh.h>
 
 /*  ----------------------------------- Link Driver */
-#include <wmd.h>
-#include <wmdioctl.h>
+#include <dspbridge/wmd.h>
+#include <dspbridge/wmdioctl.h>
 
 /*  ----------------------------------- Others */
-#include <gb.h>
+#include <dspbridge/gb.h>
 #ifdef DEBUG
-#include <uuidutil.h>
-#include <dbg.h>
+#include <dspbridge/uuidutil.h>
+#include <dspbridge/dbg.h>
 #endif
 
 /*  ----------------------------------- This */
-#include <nodepriv.h>
-#include <node.h>
+#include <dspbridge/nodepriv.h>
+#include <dspbridge/node.h>
 
 /* Static/Dynamic Loader includes */
-#include <dbll.h>
-#include <nldr.h>
+#include <dspbridge/dbll.h>
+#include <dspbridge/nldr.h>
 
 #ifndef RES_CLEANUP_DISABLE
-#include <drv.h>
-#include <drvdefs.h>
-#include <dbreg.h>
-#include <resourcecleanup.h>
+#include <dspbridge/drv.h>
+#include <dspbridge/drvdefs.h>
+#include <dspbridge/dbreg.h>
+#include <dspbridge/resourcecleanup.h>
 #endif
 
 #ifndef CONFIG_DISABLE_BRIDGE_PM
--- a/drivers/dsp/bridge/rmgr/proc.c
+++ b/drivers/dsp/bridge/rmgr/proc.c
@@ -101,52 +101,52 @@
  */
 
 /* ------------------------------------ Host OS */
-#include <host_os.h>
+#include <dspbridge/host_os.h>
 
 /*  ----------------------------------- DSP/BIOS Bridge */
-#include <std.h>
-#include <dbdefs.h>
-#include <errbase.h>
+#include <dspbridge/std.h>
+#include <dspbridge/dbdefs.h>
+#include <dspbridge/errbase.h>
 
 /*  ----------------------------------- Trace & Debug */
-#include <dbc.h>
-#include <gt.h>
+#include <dspbridge/dbc.h>
+#include <dspbridge/gt.h>
 
 /*  ----------------------------------- OS Adaptation Layer */
-#include <cfg.h>
-#include <csl.h>
-#include <list.h>
-#include <mem.h>
-#include <ntfy.h>
+#include <dspbridge/cfg.h>
+#include <dspbridge/csl.h>
+#include <dspbridge/list.h>
+#include <dspbridge/mem.h>
+#include <dspbridge/ntfy.h>
 
 /*  ----------------------------------- Mini Driver */
-#include <wmd.h>
+#include <dspbridge/wmd.h>
 
 /*  ----------------------------------- Platform Manager */
-#include <cod.h>
-#include <dev.h>
-#include <drv.h>
-#include <procpriv.h>
-#include <dmm.h>
+#include <dspbridge/cod.h>
+#include <dspbridge/dev.h>
+#include <dspbridge/drv.h>
+#include <dspbridge/procpriv.h>
+#include <dspbridge/dmm.h>
 
 /*  ----------------------------------- Resource Manager */
-#include <mgr.h>
-#include <node.h>
-#include <nldr.h>
-#include <rmm.h>
+#include <dspbridge/mgr.h>
+#include <dspbridge/node.h>
+#include <dspbridge/nldr.h>
+#include <dspbridge/rmm.h>
 
 /*  ----------------------------------- Others */
-#include <dbdcd.h>
-#include <dbreg.h>
-#include <msg.h>
-#include <wmdioctl.h>
+#include <dspbridge/dbdcd.h>
+#include <dspbridge/dbreg.h>
+#include <dspbridge/msg.h>
+#include <dspbridge/wmdioctl.h>
 
 /*  ----------------------------------- This */
-#include <proc.h>
-#include <pwr.h>
+#include <dspbridge/proc.h>
+#include <dspbridge/pwr.h>
 
 #ifndef RES_CLEANUP_DISABLE
-#include <resourcecleanup.h>
+#include <dspbridge/resourcecleanup.h>
 #endif
 #ifndef CONFIG_DISABLE_BRIDGE_PM
 #ifndef CONFIG_DISABLE_BRIDGE_DVFS
--- a/drivers/dsp/bridge/rmgr/pwr.c
+++ b/drivers/dsp/bridge/rmgr/pwr.c
@@ -32,20 +32,20 @@
  */
 
 /*  ----------------------------------- Host OS */
-#include <host_os.h>
+#include <dspbridge/host_os.h>
 
 /*  ----------------------------------- This */
-#include <pwr.h>
+#include <dspbridge/pwr.h>
 
 /*  ----------------------------------- Resource Manager */
-#include <devdefs.h>
-#include <drv.h>
+#include <dspbridge/devdefs.h>
+#include <dspbridge/drv.h>
 
 /*  ----------------------------------- Platform Manager */
-#include <dev.h>
+#include <dspbridge/dev.h>
 
 /*  ----------------------------------- Link Driver */
-#include <wmdioctl.h>
+#include <dspbridge/wmdioctl.h>
 
 /*
  *  ======== PWR_SleepDSP ========
--- a/drivers/dsp/bridge/rmgr/rmm.c
+++ b/drivers/dsp/bridge/rmgr/rmm.c
@@ -52,20 +52,20 @@
  */
 
 /*  ----------------------------------- DSP/BIOS Bridge */
-#include <std.h>
-#include <dbdefs.h>
-#include <errbase.h>
+#include <dspbridge/std.h>
+#include <dspbridge/dbdefs.h>
+#include <dspbridge/errbase.h>
 
 /*  ----------------------------------- Trace & Debug */
-#include <dbc.h>
-#include <gt.h>
+#include <dspbridge/dbc.h>
+#include <dspbridge/gt.h>
 
 /*  ----------------------------------- OS Adaptation Layer */
-#include <list.h>
-#include <mem.h>
+#include <dspbridge/list.h>
+#include <dspbridge/mem.h>
 
 /*  ----------------------------------- This */
-#include <rmm.h>
+#include <dspbridge/rmm.h>
 
 #define RMM_TARGSIGNATURE   0x544d4d52	/* "TMMR" */
 
--- a/drivers/dsp/bridge/rmgr/strm.c
+++ b/drivers/dsp/bridge/rmgr/strm.c
@@ -62,37 +62,37 @@
  */
 
 /*  ----------------------------------- Host OS */
-#include <host_os.h>
+#include <dspbridge/host_os.h>
 
 /*  ----------------------------------- DSP/BIOS Bridge */
-#include <std.h>
-#include <dbdefs.h>
-#include <errbase.h>
+#include <dspbridge/std.h>
+#include <dspbridge/dbdefs.h>
+#include <dspbridge/errbase.h>
 
 /*  ----------------------------------- Trace & Debug */
-#include <dbc.h>
-#include <gt.h>
+#include <dspbridge/dbc.h>
+#include <dspbridge/gt.h>
 
 /*  ----------------------------------- OS Adaptation Layer */
-#include <mem.h>
-#include <sync.h>
+#include <dspbridge/mem.h>
+#include <dspbridge/sync.h>
 
 /*  ----------------------------------- Mini Driver */
-#include <wmd.h>
+#include <dspbridge/wmd.h>
 
 /*  ----------------------------------- Resource Manager */
-#include <nodepriv.h>
+#include <dspbridge/nodepriv.h>
 
 /*  ----------------------------------- Others */
-#include <cmm.h>
+#include <dspbridge/cmm.h>
 
 /*  ----------------------------------- This */
-#include <strm.h>
+#include <dspbridge/strm.h>
 
 #ifndef RES_CLEANUP_DISABLE
-#include <cfg.h>
-#include <dbreg.h>
-#include <resourcecleanup.h>
+#include <dspbridge/cfg.h>
+#include <dspbridge/dbreg.h>
+#include <dspbridge/resourcecleanup.h>
 #endif
 
 /*  ----------------------------------- Defines, Data Structures, Typedefs */
--- a/drivers/dsp/bridge/services/cfg.c
+++ b/drivers/dsp/bridge/services/cfg.c
@@ -73,24 +73,24 @@
  */
 
 /*  ----------------------------------- DSP/BIOS Bridge */
-#include <std.h>
-#include <dbdefs.h>
-#include <errbase.h>
+#include <dspbridge/std.h>
+#include <dspbridge/dbdefs.h>
+#include <dspbridge/errbase.h>
 
 /*  ----------------------------------- Trace & Debug */
-#include <dbc.h>
-#include <gt.h>
+#include <dspbridge/dbc.h>
+#include <dspbridge/gt.h>
 
 /*  ----------------------------------- OS Adaptation Layer */
-#include <csl.h>
-#include <reg.h>
+#include <dspbridge/csl.h>
+#include <dspbridge/reg.h>
 
 /*  ----------------------------------- Others */
-#include <dbreg.h>
+#include <dspbridge/dbreg.h>
 
 /*  ----------------------------------- This */
-#include <cfg.h>
-#include <list.h>
+#include <dspbridge/cfg.h>
+#include <dspbridge/list.h>
 
 struct DRV_EXT {
 	struct LST_ELEM link;
--- a/drivers/dsp/bridge/services/clk.c
+++ b/drivers/dsp/bridge/services/clk.c
@@ -33,24 +33,24 @@
  */
 
 /*  ----------------------------------- Host OS */
-#include <host_os.h>
+#include <dspbridge/host_os.h>
 
 /*  ----------------------------------- DSP/BIOS Bridge */
-#include <std.h>
-#include <dbdefs.h>
-#include <errbase.h>
+#include <dspbridge/std.h>
+#include <dspbridge/dbdefs.h>
+#include <dspbridge/errbase.h>
 
 /*  ----------------------------------- Trace & Debug */
-#include <dbc.h>
-#include <gt.h>
+#include <dspbridge/dbc.h>
+#include <dspbridge/gt.h>
 
 /*  ----------------------------------- OS Adaptation Layer */
-#include <csl.h>
-#include <mem.h>
+#include <dspbridge/csl.h>
+#include <dspbridge/mem.h>
 
 /*  ----------------------------------- This */
-#include <clk.h>
-#include <util.h>
+#include <dspbridge/clk.h>
+#include <dspbridge/util.h>
 
 
 /*  ----------------------------------- Defines, Data Structures, Typedefs */
--- a/drivers/dsp/bridge/services/csl.c
+++ b/drivers/dsp/bridge/services/csl.c
@@ -51,18 +51,18 @@
  */
 
 /* ----------------------------------- Host OS */
-#include <host_os.h>
+#include <dspbridge/host_os.h>
 
 /*  ----------------------------------- DSP/BIOS Bridge */
-#include <std.h>
-#include <dbdefs.h>
+#include <dspbridge/std.h>
+#include <dspbridge/dbdefs.h>
 
 /*  ----------------------------------- Trace & Debug */
-#include <dbc.h>
-#include <gt.h>
+#include <dspbridge/dbc.h>
+#include <dspbridge/gt.h>
 
 /*  ----------------------------------- This */
-#include <csl.h>
+#include <dspbridge/csl.h>
 
 /* Is character c in the string pstrDelim? */
 #define IsDelimiter(c, pstrDelim) ((c != '\0') && \
--- a/drivers/dsp/bridge/services/dbg.c
+++ b/drivers/dsp/bridge/services/dbg.c
@@ -46,16 +46,16 @@
  */
 
 /*  ----------------------------------- DSP/BIOS Bridge */
-#include <std.h>
-#include <dbdefs.h>
-#include <errbase.h>
+#include <dspbridge/std.h>
+#include <dspbridge/dbdefs.h>
+#include <dspbridge/errbase.h>
 
 /*  ----------------------------------- Trace & Debug */
-#include <dbc.h>
-#include <gt.h>
+#include <dspbridge/dbc.h>
+#include <dspbridge/gt.h>
 
 /*  ----------------------------------- This */
-#include <dbg.h>
+#include <dspbridge/dbg.h>
 
 /*  ----------------------------------- Globals */
 #if GT_TRACE
--- a/drivers/dsp/bridge/services/dpc.c
+++ b/drivers/dsp/bridge/services/dpc.c
@@ -45,22 +45,22 @@
  */
 
 /*  ----------------------------------- Host OS */
-#include <host_os.h>
+#include <dspbridge/host_os.h>
 
 /*  ----------------------------------- DSP/BIOS Bridge */
-#include <std.h>
-#include <dbdefs.h>
-#include <errbase.h>
+#include <dspbridge/std.h>
+#include <dspbridge/dbdefs.h>
+#include <dspbridge/errbase.h>
 
 /*  ----------------------------------- Trace & Debug */
-#include <dbc.h>
-#include <gt.h>
+#include <dspbridge/dbc.h>
+#include <dspbridge/gt.h>
 
 /*  ----------------------------------- OS Adaptation Layer */
-#include <mem.h>
+#include <dspbridge/mem.h>
 
 /*  ----------------------------------- This */
-#include <dpc.h>
+#include <dspbridge/dpc.h>
 
 /*  ----------------------------------- Defines, Data Structures, Typedefs */
 #define SIGNATURE       0x5f435044	/* "DPC_" (in reverse). */
--- a/drivers/dsp/bridge/services/kfile.c
+++ b/drivers/dsp/bridge/services/kfile.c
@@ -42,23 +42,23 @@
  */
 
 /*  ----------------------------------- Host OS */
-#include <host_os.h>
+#include <dspbridge/host_os.h>
 
 /*  ----------------------------------- DSP/BIOS Bridge */
-#include <std.h>
-#include <dbdefs.h>
-#include <errbase.h>
+#include <dspbridge/std.h>
+#include <dspbridge/dbdefs.h>
+#include <dspbridge/errbase.h>
 
 /*  ----------------------------------- Trace & Debug */
-#include <dbc.h>
-#include <gt.h>
+#include <dspbridge/dbc.h>
+#include <dspbridge/gt.h>
 
 /*  ----------------------------------- OS Adaptation Layer */
-#include <csl.h>
-#include <mem.h>
+#include <dspbridge/csl.h>
+#include <dspbridge/mem.h>
 
 /*  ----------------------------------- This */
-#include <kfile.h>
+#include <dspbridge/kfile.h>
 
 /*  ----------------------------------- Defines, Data Structures, Typedefs */
 #define SIGNATURE           0x4c49464b	/* hex code of KFIL (reversed) */
--- a/drivers/dsp/bridge/services/list.c
+++ b/drivers/dsp/bridge/services/list.c
@@ -48,18 +48,18 @@
  */
 
 /*  ----------------------------------- DSP/BIOS Bridge */
-#include <std.h>
-#include <dbdefs.h>
+#include <dspbridge/std.h>
+#include <dspbridge/dbdefs.h>
 
 /*  ----------------------------------- Trace & Debug */
-#include <dbc.h>
-#include <gt.h>
+#include <dspbridge/dbc.h>
+#include <dspbridge/gt.h>
 
 /*  ----------------------------------- OS Adaptation Layer */
-#include <mem.h>
+#include <dspbridge/mem.h>
 
 /*  ----------------------------------- This */
-#include <list.h>
+#include <dspbridge/list.h>
 
 /*  ----------------------------------- Globals */
 #if GT_TRACE
--- a/drivers/dsp/bridge/services/mem.c
+++ b/drivers/dsp/bridge/services/mem.c
@@ -52,20 +52,20 @@
  */
 
 /*  ----------------------------------- Host OS */
-#include <host_os.h>
+#include <dspbridge/host_os.h>
 
 /*  ----------------------------------- DSP/BIOS Bridge */
-#include <std.h>
-#include <dbdefs.h>
-#include <errbase.h>
+#include <dspbridge/std.h>
+#include <dspbridge/dbdefs.h>
+#include <dspbridge/errbase.h>
 
 /*  ----------------------------------- Trace & Debug */
-#include <dbc.h>
-#include <gt.h>
+#include <dspbridge/dbc.h>
+#include <dspbridge/gt.h>
 
 /*  ----------------------------------- This */
-#include <mem.h>
-#include <list.h>
+#include <dspbridge/mem.h>
+#include <dspbridge/list.h>
 
 /*  ----------------------------------- Defines */
 #define MEM_512MB   0x1fffffff
--- a/drivers/dsp/bridge/services/ntfy.c
+++ b/drivers/dsp/bridge/services/ntfy.c
@@ -40,25 +40,25 @@
  */
 
 /*  ----------------------------------- Host OS */
-#include <host_os.h>
+#include <dspbridge/host_os.h>
 
 /*  ----------------------------------- DSP/BIOS Bridge */
-#include <std.h>
-#include <dbdefs.h>
-#include <errbase.h>
+#include <dspbridge/std.h>
+#include <dspbridge/dbdefs.h>
+#include <dspbridge/errbase.h>
 
 /*  ----------------------------------- Trace & Debug */
-#include <dbc.h>
-#include <gt.h>
+#include <dspbridge/dbc.h>
+#include <dspbridge/gt.h>
 
 /*  ----------------------------------- OS Adaptation Layer */
-#include <csl.h>
-#include <list.h>
-#include <mem.h>
-#include <sync.h>
+#include <dspbridge/csl.h>
+#include <dspbridge/list.h>
+#include <dspbridge/mem.h>
+#include <dspbridge/sync.h>
 
 /*  ----------------------------------- This */
-#include <ntfy.h>
+#include <dspbridge/ntfy.h>
 
 /*  ----------------------------------- Defines, Data Structures, Typedefs */
 #define NTFY_SIGNATURE      0x5946544e	/* "YFTN" */
--- a/drivers/dsp/bridge/services/reg.c
+++ b/drivers/dsp/bridge/services/reg.c
@@ -34,26 +34,26 @@
  */
 
 /*  ----------------------------------- Host OS */
-#include <host_os.h>
+#include <dspbridge/host_os.h>
 
 /*  ----------------------------------- DSP/BIOS Bridge */
-#include <std.h>
-#include <dbdefs.h>
-#include <errbase.h>
+#include <dspbridge/std.h>
+#include <dspbridge/dbdefs.h>
+#include <dspbridge/errbase.h>
 
 /*  ----------------------------------- Trace & Debug */
-#include <dbc.h>
-#include <gt.h>
+#include <dspbridge/dbc.h>
+#include <dspbridge/gt.h>
 
 /*  ----------------------------------- OS Adaptation Layer */
-#include <csl.h>
-#include <mem.h>
+#include <dspbridge/csl.h>
+#include <dspbridge/mem.h>
 
 /*  ----------------------------------- Others */
-#include <dbreg.h>
+#include <dspbridge/dbreg.h>
 
 /*  ----------------------------------- This */
-#include <reg.h>
+#include <dspbridge/reg.h>
 #include <regsup.h>
 
 #if GT_TRACE
--- a/drivers/dsp/bridge/services/regsup.c
+++ b/drivers/dsp/bridge/services/regsup.c
@@ -29,21 +29,21 @@
  */
 
 /*  ----------------------------------- Host OS */
-#include <host_os.h>
+#include <dspbridge/host_os.h>
 
 /*  ----------------------------------- DSP/BIOS Bridge */
-#include <std.h>
-#include <dbdefs.h>
-#include <errbase.h>
-#include <dbreg.h>
+#include <dspbridge/std.h>
+#include <dspbridge/dbdefs.h>
+#include <dspbridge/errbase.h>
+#include <dspbridge/dbreg.h>
 
 /*  ----------------------------------- Trace & Debug */
-#include <dbc.h>
-#include <gt.h>
+#include <dspbridge/dbc.h>
+#include <dspbridge/gt.h>
 
 /*  ----------------------------------- OS Adaptation Layer */
-#include <mem.h>
-#include <csl.h>
+#include <dspbridge/mem.h>
+#include <dspbridge/csl.h>
 
 /*  ----------------------------------- This */
 #include <regsup.h>
--- a/drivers/dsp/bridge/services/services.c
+++ b/drivers/dsp/bridge/services/services.c
@@ -32,32 +32,32 @@
  *! 01-Feb-2000 kc: Created.
  */
 
-#include <host_os.h>
+#include <dspbridge/host_os.h>
 
 /*  ----------------------------------- DSP/BIOS Bridge */
-#include <std.h>
-#include <dbdefs.h>
+#include <dspbridge/std.h>
+#include <dspbridge/dbdefs.h>
 
 /*  ----------------------------------- Trace & Debug */
-#include <dbc.h>
-#include <gt.h>
+#include <dspbridge/dbc.h>
+#include <dspbridge/gt.h>
 
 /*  ----------------------------------- OS Adaptation Layer */
-#include <cfg.h>
-#include <csl.h>
-#include <dbg.h>
-#include <dpc.h>
-#include <kfile.h>
-#include <list.h>
-#include <mem.h>
-#include <ntfy.h>
-#include <reg.h>
-#include <sync.h>
-#include <clk.h>
-#include <util.h>
+#include <dspbridge/cfg.h>
+#include <dspbridge/csl.h>
+#include <dspbridge/dbg.h>
+#include <dspbridge/dpc.h>
+#include <dspbridge/kfile.h>
+#include <dspbridge/list.h>
+#include <dspbridge/mem.h>
+#include <dspbridge/ntfy.h>
+#include <dspbridge/reg.h>
+#include <dspbridge/sync.h>
+#include <dspbridge/clk.h>
+#include <dspbridge/util.h>
 
 /*  ----------------------------------- This */
-#include <services.h>
+#include <dspbridge/services.h>
 
 /*  ----------------------------------- Globals */
 #if GT_TRACE
--- a/drivers/dsp/bridge/services/sync.c
+++ b/drivers/dsp/bridge/services/sync.c
@@ -48,23 +48,23 @@
  */
 
 /*  ----------------------------------- Host OS */
-#include <host_os.h>
+#include <dspbridge/host_os.h>
 
 /*  ----------------------------------- DSP/BIOS Bridge */
-#include <std.h>
-#include <dbdefs.h>
-#include <errbase.h>
+#include <dspbridge/std.h>
+#include <dspbridge/dbdefs.h>
+#include <dspbridge/errbase.h>
 
 /*  ----------------------------------- Trace & Debug */
-#include <dbc.h>
-#include <gt.h>
+#include <dspbridge/dbc.h>
+#include <dspbridge/gt.h>
 
 /*  ----------------------------------- OS Adaptation Layer */
-#include <csl.h>
-#include <mem.h>
+#include <dspbridge/csl.h>
+#include <dspbridge/mem.h>
 
 /*  ----------------------------------- This */
-#include <sync.h>
+#include <dspbridge/sync.h>
 
 /*  ----------------------------------- Defines, Data Structures, Typedefs */
 #define SIGNATURE       0x434e5953	/* "SYNC" (in reverse) */
--- a/drivers/dsp/bridge/wmd/_deh.h
+++ b/drivers/dsp/bridge/wmd/_deh.h
@@ -28,9 +28,9 @@
 #ifndef _DEH_
 #define _DEH_
 
-#include <dpc.h>
-#include <ntfy.h>
-#include <wmd.h>
+#include <dspbridge/dpc.h>
+#include <dspbridge/ntfy.h>
+#include <dspbridge/wmd.h>
 
 #define SIGNATURE       0x5f484544	/* "DEH_" backwards */
 
--- a/drivers/dsp/bridge/wmd/_msg_sm.h
+++ b/drivers/dsp/bridge/wmd/_msg_sm.h
@@ -35,8 +35,8 @@
 #ifndef _MSG_SM_
 #define _MSG_SM_
 
-#include <list.h>
-#include <msgdefs.h>
+#include <dspbridge/list.h>
+#include <dspbridge/msgdefs.h>
 
 /*
  *  These target side symbols define the beginning and ending addresses
--- a/drivers/dsp/bridge/wmd/_tiomap.h
+++ b/drivers/dsp/bridge/wmd/_tiomap.h
@@ -46,12 +46,12 @@
 #ifndef _TIOMAP_
 #define _TIOMAP_
 
-#include <devdefs.h>
+#include <dspbridge/devdefs.h>
 #include <hw_defs.h>
 #include <hw_mbox.h>
-#include <wmdioctl.h>		/* for WMDIOCTL_EXTPROC defn */
-#include "sync.h"
-#include "clk.h"
+#include <dspbridge/wmdioctl.h>		/* for WMDIOCTL_EXTPROC defn */
+#include <dspbridge/sync.h>
+#include <dspbridge/clk.h>
 
 struct MAP_L4PERIPHERAL {
 	u32 physAddr;
--- a/drivers/dsp/bridge/wmd/chnl_sm.c
+++ b/drivers/dsp/bridge/wmd/chnl_sm.c
@@ -114,32 +114,32 @@
  */
 
 /*  ----------------------------------- OS */
-#include <host_os.h>
+#include <dspbridge/host_os.h>
 
 /*  ----------------------------------- DSP/BIOS Bridge */
-#include <std.h>
-#include <dbdefs.h>
-#include <errbase.h>
+#include <dspbridge/std.h>
+#include <dspbridge/dbdefs.h>
+#include <dspbridge/errbase.h>
 
 /*  ----------------------------------- Trace & Debug */
-#include <dbc.h>
-#include <dbg.h>
+#include <dspbridge/dbc.h>
+#include <dspbridge/dbg.h>
 
 /*  ----------------------------------- OS Adaptation Layer */
-#include <mem.h>
-#include <cfg.h>
-#include <csl.h>
-#include <sync.h>
+#include <dspbridge/mem.h>
+#include <dspbridge/cfg.h>
+#include <dspbridge/csl.h>
+#include <dspbridge/sync.h>
 
 /*  ----------------------------------- Mini-Driver */
-#include <wmd.h>
-#include <wmdchnl.h>
+#include <dspbridge/wmd.h>
+#include <dspbridge/wmdchnl.h>
 
 /*  ----------------------------------- Platform Manager */
-#include <dev.h>
+#include <dspbridge/dev.h>
 
 /*  ----------------------------------- Others */
-#include <io_sm.h>
+#include <dspbridge/io_sm.h>
 
 /*  ----------------------------------- Define for This */
 #define USERMODE_ADDR   PAGE_OFFSET
--- a/drivers/dsp/bridge/wmd/io_sm.c
+++ b/drivers/dsp/bridge/wmd/io_sm.c
@@ -110,7 +110,7 @@
  */
 
 /*  ----------------------------------- Host OS */
-#include <host_os.h>
+#include <dspbridge/host_os.h>
 #ifndef CONFIG_DISABLE_BRIDGE_PM
 #ifndef CONFIG_DISABLE_BRIDGE_DVFS
 #ifndef CONFIG_OMAP3_PM
@@ -122,50 +122,50 @@
 #endif
 
 /*  ----------------------------------- DSP/BIOS Bridge */
-#include <std.h>
-#include <dbdefs.h>
-#include <errbase.h>
+#include <dspbridge/std.h>
+#include <dspbridge/dbdefs.h>
+#include <dspbridge/errbase.h>
 
 /*  ----------------------------------- Trace & Debug */
-#include <dbc.h>
-#include <dbg.h>
+#include <dspbridge/dbc.h>
+#include <dspbridge/dbg.h>
 
 /*  ----------------------------------- OS Adaptation Layer */
-#include <cfg.h>
-#include <dpc.h>
-#include <mem.h>
-#include <ntfy.h>
-#include <sync.h>
-#include <reg.h>
+#include <dspbridge/cfg.h>
+#include <dspbridge/dpc.h>
+#include <dspbridge/mem.h>
+#include <dspbridge/ntfy.h>
+#include <dspbridge/sync.h>
+#include <dspbridge/reg.h>
 
 /* ------------------------------------ Hardware Abstraction Layer */
 #include <hw_defs.h>
 #include <hw_mmu.h>
 
 /*  ----------------------------------- Mini Driver */
-#include <wmddeh.h>
-#include <wmdio.h>
-#include <wmdioctl.h>
+#include <dspbridge/wmddeh.h>
+#include <dspbridge/wmdio.h>
+#include <dspbridge/wmdioctl.h>
 #include <_tiomap.h>
 #include <tiomap_io.h>
 #include <_tiomap_pwr.h>
 #include <tiomap_io.h>
 
 /*  ----------------------------------- Platform Manager */
-#include <cod.h>
-#include <dev.h>
-#include <chnl_sm.h>
-#include <dbreg.h>
+#include <dspbridge/cod.h>
+#include <dspbridge/dev.h>
+#include <dspbridge/chnl_sm.h>
+#include <dspbridge/dbreg.h>
 
 /*  ----------------------------------- Others */
-#include <rms_sh.h>
-#include <mgr.h>
+#include <dspbridge/rms_sh.h>
+#include <dspbridge/mgr.h>
 #include "_cmm.h"
 
 /*  ----------------------------------- This */
-#include <io_sm.h>
+#include <dspbridge/io_sm.h>
 #include "_msg_sm.h"
-#include <gt.h>
+#include <dspbridge/gt.h>
 
 /*  ----------------------------------- Defines, Data Structures, Typedefs */
 #define OUTPUTNOTREADY  0xffff
--- a/drivers/dsp/bridge/wmd/mmu_fault.c
+++ b/drivers/dsp/bridge/wmd/mmu_fault.c
@@ -32,22 +32,22 @@
  */
 
 /*  ----------------------------------- DSP/BIOS Bridge */
-#include <std.h>
-#include <dbdefs.h>
-#include <errbase.h>
+#include <dspbridge/std.h>
+#include <dspbridge/dbdefs.h>
+#include <dspbridge/errbase.h>
 
 /*  ----------------------------------- Trace & Debug */
-#include <host_os.h>
-#include <dbc.h>
-#include <dbg.h>
+#include <dspbridge/host_os.h>
+#include <dspbridge/dbc.h>
+#include <dspbridge/dbg.h>
 
 /*  ----------------------------------- OS Adaptation Layer */
-#include <dpc.h>
-#include <mem.h>
-#include <drv.h>
+#include <dspbridge/dpc.h>
+#include <dspbridge/mem.h>
+#include <dspbridge/drv.h>
 
 /*  ----------------------------------- Link Driver */
-#include <wmddeh.h>
+#include <dspbridge/wmddeh.h>
 
 /* ------------------------------------ Hardware Abstraction Layer */
 #include <hw_defs.h>
@@ -55,7 +55,7 @@
 
 /*  ----------------------------------- This */
 #include "_deh.h"
-#include <cfg.h>
+#include <dspbridge/cfg.h>
 #include "_tiomap_mmu.h"
 #include "_tiomap.h"
 #include "mmu_fault.h"
--- a/drivers/dsp/bridge/wmd/msg_sm.c
+++ b/drivers/dsp/bridge/wmd/msg_sm.c
@@ -43,27 +43,27 @@
  */
 
 /*  ----------------------------------- DSP/BIOS Bridge */
-#include <std.h>
-#include <dbdefs.h>
-#include <errbase.h>
+#include <dspbridge/std.h>
+#include <dspbridge/dbdefs.h>
+#include <dspbridge/errbase.h>
 
 /*  ----------------------------------- Trace & Debug */
-#include <dbc.h>
+#include <dspbridge/dbc.h>
 
 /*  ----------------------------------- OS Adaptation Layer */
-#include <list.h>
-#include <mem.h>
-#include <sync.h>
+#include <dspbridge/list.h>
+#include <dspbridge/mem.h>
+#include <dspbridge/sync.h>
 
 /*  ----------------------------------- Platform Manager */
-#include <dev.h>
+#include <dspbridge/dev.h>
 
 /*  ----------------------------------- Others */
-#include <io_sm.h>
+#include <dspbridge/io_sm.h>
 
 /*  ----------------------------------- This */
 #include <_msg_sm.h>
-#include <wmdmsg.h>
+#include <dspbridge/wmdmsg.h>
 
 /*  ----------------------------------- Defines, Data Structures, Typedefs */
 #define MSGQ_SIGNATURE      0x5147534d	/* "QGSM" */
--- a/drivers/dsp/bridge/wmd/tiomap3430.c
+++ b/drivers/dsp/bridge/wmd/tiomap3430.c
@@ -27,26 +27,26 @@
  */
 
 /*  ----------------------------------- Host OS */
-#include <host_os.h>
+#include <dspbridge/host_os.h>
 
 /*  ----------------------------------- DSP/BIOS Bridge */
-#include <std.h>
-#include <dbdefs.h>
-#include <errbase.h>
+#include <dspbridge/std.h>
+#include <dspbridge/dbdefs.h>
+#include <dspbridge/errbase.h>
 
 /*  ----------------------------------- Trace & Debug */
-#include <dbc.h>
-#include <dbg.h>
+#include <dspbridge/dbc.h>
+#include <dspbridge/dbg.h>
 
 /*  ----------------------------------- OS Adaptation Layer */
-#include <mem.h>
-#include <util.h>
-#include <reg.h>
-#include <dbreg.h>
-#include <cfg.h>
-#include <drv.h>
-#include <csl.h>
-#include <sync.h>
+#include <dspbridge/mem.h>
+#include <dspbridge/util.h>
+#include <dspbridge/reg.h>
+#include <dspbridge/dbreg.h>
+#include <dspbridge/cfg.h>
+#include <dspbridge/drv.h>
+#include <dspbridge/csl.h>
+#include <dspbridge/sync.h>
 
 /* ------------------------------------ Hardware Abstraction Layer */
 #include <hw_defs.h>
@@ -56,19 +56,19 @@
 #include <hw_mbox.h>
 
 /*  ----------------------------------- Link Driver */
-#include <wmd.h>
-#include <wmdchnl.h>
-#include <wmddeh.h>
-#include <wmdio.h>
-#include <wmdmsg.h>
-#include <pwr.h>
-#include <chnl_sm.h>
-#include <io_sm.h>
+#include <dspbridge/wmd.h>
+#include <dspbridge/wmdchnl.h>
+#include <dspbridge/wmddeh.h>
+#include <dspbridge/wmdio.h>
+#include <dspbridge/wmdmsg.h>
+#include <dspbridge/pwr.h>
+#include <dspbridge/chnl_sm.h>
+#include <dspbridge/io_sm.h>
 
 /*  ----------------------------------- Platform Manager */
-#include <dev.h>
-#include <wcd.h>
-#include <dmm.h>
+#include <dspbridge/dev.h>
+#include <dspbridge/wcd.h>
+#include <dspbridge/dmm.h>
 
 /*  ----------------------------------- Local */
 #include "_tiomap.h"
--- a/drivers/dsp/bridge/wmd/tiomap3430_pwr.c
+++ b/drivers/dsp/bridge/wmd/tiomap3430_pwr.c
@@ -30,24 +30,24 @@
  */
 
 /*  ----------------------------------- DSP/BIOS Bridge */
-#include <dbdefs.h>
-#include <errbase.h>
-#include <cfg.h>
-#include <drv.h>
-#include <io_sm.h>
-#include <chnl_sm.h>
+#include <dspbridge/dbdefs.h>
+#include <dspbridge/errbase.h>
+#include <dspbridge/cfg.h>
+#include <dspbridge/drv.h>
+#include <dspbridge/io_sm.h>
+#include <dspbridge/chnl_sm.h>
 
 /*  ----------------------------------- Trace & Debug */
-#include <dbg.h>
+#include <dspbridge/dbg.h>
 
 /*  ----------------------------------- OS Adaptation Layer */
-#include <mem.h>
-#include <util.h>
+#include <dspbridge/mem.h>
+#include <dspbridge/util.h>
 
 /*  ----------------------------------- Platform Manager */
-#include <brddefs.h>
-#include <dev.h>
-#include <iodefs.h>
+#include <dspbridge/brddefs.h>
+#include <dspbridge/dev.h>
+#include <dspbridge/iodefs.h>
 
 /* ------------------------------------ Hardware Abstraction Layer */
 #include <hw_defs.h>
@@ -55,7 +55,7 @@
 #include <hw_prcm.h>
 #include <hw_mmu.h>
 
-#include <pwr_sh.h>
+#include <dspbridge/pwr_sh.h>
 
 /*  ----------------------------------- specific to this file */
 #include "_tiomap.h"
--- a/drivers/dsp/bridge/wmd/tiomap_io.c
+++ b/drivers/dsp/bridge/wmd/tiomap_io.c
@@ -30,21 +30,21 @@
  */
 
 /*  ----------------------------------- DSP/BIOS Bridge */
-#include <dbdefs.h>
-#include <errbase.h>
+#include <dspbridge/dbdefs.h>
+#include <dspbridge/errbase.h>
 
 /*  ----------------------------------- Trace & Debug */
-#include <dbc.h>
-#include <dbg.h>
+#include <dspbridge/dbc.h>
+#include <dspbridge/dbg.h>
 
 /*  ----------------------------------- Platform Manager */
-#include <dev.h>
-#include <drv.h>
+#include <dspbridge/dev.h>
+#include <dspbridge/drv.h>
 
 /*  ----------------------------------- OS Adaptation Layer */
-#include <mem.h>
-#include <util.h>
-#include <cfg.h>
+#include <dspbridge/mem.h>
+#include <dspbridge/util.h>
+#include <dspbridge/cfg.h>
 
 /*  ----------------------------------- specific to this file */
 #include "_tiomap.h"
--- a/drivers/dsp/bridge/wmd/tiomap_sm.c
+++ b/drivers/dsp/bridge/wmd/tiomap_sm.c
@@ -37,26 +37,26 @@
  */
 
 /*  ----------------------------------- Host OS */
-#include <host_os.h>
+#include <dspbridge/host_os.h>
 /*  ----------------------------------- DSP/BIOS Bridge */
-#include <std.h>
-#include <dbdefs.h>
-#include <errbase.h>
+#include <dspbridge/std.h>
+#include <dspbridge/dbdefs.h>
+#include <dspbridge/errbase.h>
 
 /*  ----------------------------------- Trace & Debug */
-#include <dbg.h>
+#include <dspbridge/dbg.h>
 
 /*  ----------------------------------- OS Adaptation Layer */
-#include <cfg.h>
-#include <drv.h>
-#include <util.h>
+#include <dspbridge/cfg.h>
+#include <dspbridge/drv.h>
+#include <dspbridge/util.h>
 
 /*  ----------------------------------- Mini Driver */
-#include <wmd.h>
+#include <dspbridge/wmd.h>
 
 /*  ----------------------------------- Platform Manager */
-#include <wcd.h>
-#include <dev.h>
+#include <dspbridge/wcd.h>
+#include <dspbridge/dev.h>
 
 /* ------------------------------------ Hardware Abstraction Layer */
 #include <hw_defs.h>
@@ -64,7 +64,7 @@
 
 /*  ----------------------------------- This */
 #include "_tiomap.h"
-#include <chnl_sm.h>
+#include <dspbridge/chnl_sm.h>
 #include "_tiomap_pwr.h"
 
 #ifndef CONFIG_DISABLE_BRIDGE_PM
--- a/drivers/dsp/bridge/wmd/ue_deh.c
+++ b/drivers/dsp/bridge/wmd/ue_deh.c
@@ -32,31 +32,31 @@
  */
 
 /*  ----------------------------------- Host OS */
-#include <host_os.h>
+#include <dspbridge/host_os.h>
 
 /*  ----------------------------------- DSP/BIOS Bridge */
-#include <std.h>
-#include <dbdefs.h>
-#include <errbase.h>
+#include <dspbridge/std.h>
+#include <dspbridge/dbdefs.h>
+#include <dspbridge/errbase.h>
 
 /*  ----------------------------------- Trace & Debug */
-#include <dbc.h>
-#include <dbg.h>
+#include <dspbridge/dbc.h>
+#include <dspbridge/dbg.h>
 
 /*  ----------------------------------- OS Adaptation Layer */
-#include <csl.h>
-#include <cfg.h>
-#include <dpc.h>
-#include <mem.h>
-#include <ntfy.h>
-#include <drv.h>
+#include <dspbridge/csl.h>
+#include <dspbridge/cfg.h>
+#include <dspbridge/dpc.h>
+#include <dspbridge/mem.h>
+#include <dspbridge/ntfy.h>
+#include <dspbridge/drv.h>
 
 /*  ----------------------------------- Link Driver */
-#include <wmddeh.h>
+#include <dspbridge/wmddeh.h>
 
 /*  ----------------------------------- Platform Manager */
-#include <dev.h>
-#include <wcd.h>
+#include <dspbridge/dev.h>
+#include <dspbridge/wcd.h>
 
 /* ------------------------------------ Hardware Abstraction Layer */
 #include <hw_defs.h>
@@ -68,7 +68,7 @@
 #include "_deh.h"
 #include "_tiomap_mmu.h"
 #include "_tiomap_pwr.h"
-#include <io_sm.h>
+#include <dspbridge/io_sm.h>
 
 static struct HW_MMUMapAttrs_t  mapAttrs = { HW_LITTLE_ENDIAN,
 					HW_ELEM_SIZE_16BIT,
